
\chapter{Introduction\label{ch:intro}}

As an undergraduate genetics major I became fascinated with complex traits and diseases that are collectively shaped by many genetic variants and environmental factors.  While the genetics of these traits is of huge societal importance whether we are concerned with corn yield or heart attack risk, connecting quantitative traits to their genetic determinants has been notoriously difficult \textcolor{red}{missing heritability}.  As I began at Princeton, I wanted to better understand the source of missing heritability and how the explanatory insufficiency of GWAS may be better addressed. My insight into this problem was that the gulf separating genetic variation and whole-body complex traits is the massive complexity of organismal and cellular physiology.

At the scale of cellular and organism-level systems, thousands of genes and environmental effects collide leading to non-additive interactions, such as those resulting in canalization. Because the outputs of these systems cannot be explained using additive genetics, the combination of many such sub-systems into organism-level phenotypes will only further distort the relationship between individual genetic variants and overall phenotypic variability. While this characterization of the problem is hard to dispute, if non-additive genetics strongly shaped phenotypic variation then we would be left as at an impasse because no genome-scale method could have the power to rigorously detect such interactions (without assuming a significant main effect). An alternative to this approach can be found in systems biology, where researchers continually build bottom-up models of systems operations based on a relatively small number of inputs.

A single systems biology model may focus on a minute aspect of an organism, such as how the expression of a single gene, YFG, is determined based on the activity of its primary transcriptional repressor and cis-regulatory variation in YFG. Such a simple model of YFG expression can in turn be incorporated into a model of YFG protein abundance and this can be built into a model of pathway activity.  The importance of each model is that it integrates information within the appropriate context of the systems operation, allowing changes in system output (and uncertainty) to be expressed as a non-linear function of changes in inputs. In a perfect world where systems are well understood, genetic and environmental variants could be propagated through individual subsystems, appropriately aggregating information and accurately predicting quantitative traits/diseases, without requiring extensive measurement of an individuals. This utopian vision of systems biology is quite removed from reality, as model building in systems biology requires extensive experimentation and we generally have limited ability to identify which inputs are sufficient to characterize a given system.

The need for extensive experimentation and challenges in model discovery lay at odds in experimental design, as given fixed resources, there are tradeoffs between producing an experiment which measures all of the desired variables and characterizing a sufficient number of conditions (and replicates) to distill anything meaningful from the quantitative information. This problem is further exacerbated as funding is further distributed among groups which seek to characterize different systems, in different organisms. 

Focused on yeast as a platform where diverse physiological conditions can be generated and is readily amenable to experimental measurements that can be used simultaneously to address diverse questions. Through combined measurements of transcript, protein and metabolite abundances as well as determining the rates of cellular fluxes across 25 distinct growth conditions. I used this multi-omic framework to address two major questions in systems biology, how regulation establishes levels of functional proteins how the proteome and environment interact to construct metabolism.

The analysis of high-dimensional data, naturally entails the use of appropriate, principled methods, the subject of chapter \ref{ch:quant_anal}. Individual datasets can provide a glance at one aspect of behavior, but they are limited due to an incomplete perspective.  In chapter \ref{ch:simmer}, I consider the interaction of enzyme regulation and environmental variation in shaping metabolism. Chapter \ref{ch:ptcor} focuses on studying the interface between two datasets to determine the extent to which transcriptional changes propagate into meaningful changes in protein abundance and investigating the nature of their departures.  


\begin{figure}[h!]
\begin{center}
\includegraphics[]{ch-intro/Figures/intermediatePhenotypes.pdf}
\end{center}
\caption{woot}
\label{fig:intpheno}
\end{figure}

\begin{outline}
\1 High-throughput phenotyping is sufficient to quantify many of the key players
\2 Which to do ? transcripts, proteins, metabolites, phosphorylation events ..
\1 Systems biology
\2 If we have a snap-shot of biology from multiple angles, this seems like a good things, but how can this information be used intelligently
\2 Bottom-up modeling of the phenotypic covariation
\3 Result is correlated with the source
\3 Generally when many species are considered, this is not sufficient because  
\2 The space of possible interactions is huge
\3 Unless we have a preponderance of data, we must rationally lay our data onto the scaffold of existing knowledge
\1 genetic variation in metabolism
\1 Metabolism is relatively robust to variation
\end{outline}


