
\chapter{Introduction\label{ch:intro}}

As an undergraduate genetics major I became fascinated with complex traits and diseases such as height, corn yield and heart attack risk. While these traits are of huge societal importance, each is collectively determined by numerous genetic variants and environmental factors and consequently relating quantitative traits to their genetic determinants has been notoriously difficult \cite{Manolio:2009jp}. As I began at Princeton, I wanted to better understand the source of missing heritability and how the explanatory insufficiency of the GWAS paradigm may be better addressed. My insight into this problem was that the gulf separating genetic variation and whole-body complex traits is the massive complexity of organismal and cellular physiology.

At the scale of cellular and organism-level systems, thousands of genes and environmental effects collide leading to non-additive, non-linear interactions, such as those resulting in canalization \cite{Waddington:1942wy}. Because the outputs of these systems cannot be explained using additive genetics, the combination of many such sub-systems into organism-level phenotypes will only further distort the relationship between individual genetic variants and overall phenotypic variability. 
Outside of tightly controlled studies of microbial growth \cite{Bloom:2013bq} it is likely that additive genetics can only explain a minority of phenotypic variation regardless of statistical power. Furthermore, genome-scale methods which attempt to account for such non-additive and non-linear interactions will be unable to rigorously detect such interactions because of the massive space of such interactions \cite{Friedman:1997kn}. An alternative to this unsupervised approach can be found in systems biology, where researchers continually build bottom-up models of systems operations based on a relatively small number of inputs.

\subsection{Harnessing the power of systems biology}

A single systems biology model may focus on a minute aspect of an organism, such as how the expression of a single gene, your favorite gene (YFG), is determined based on the activity of its primary transcriptional repressor and cis-regulatory variation in YFG \cite{Nuzhdin:2012ii}. Such a simple model of YFG expression can be used to characterize YFG protein abundance \cite{Jovanovic:2015hp}, which can in turn be built into a model of signaling or metabolic pathway activity \cite{Neves:2002bk, Chassagnole:2002ty}.  The importance of each model is that it integrates information within the appropriate context of the systems operation, allowing changes in system output (and uncertainty) to be expressed as a non-linear function of variation in inputs.

In a perfect world where systems are well understood, genetic and environmental variants could be propagated through individual subsystems, appropriately aggregating information and ultimately allowing for accurate prediction of quantitative traits and disease risk.  Importantly, at such a juncture, prediction would require far less data than would be necessary to initially parameterize the system. This utopian vision of systems biology is quite removed from reality, as model building in systems biology requires extensive experimentation and we generally have limited ability to identify which inputs are sufficient to characterize a given system.

The need for extensive experimentation and challenges in model discovery lay at odds in experimental design, as given fixed resources, there are tradeoffs between producing an experiment which measures all of the desired variables and characterizing a sufficient number of conditions (and replicates) to distill anything meaningful from the quantitative information. This problem is further exacerbated as funding is further distributed among groups which seek to characterize different systems, in different organisms, with limited direct cross-domain translatability. 

To simplify the complex model inference problems of systems biology, I adopted an experimentally-guided reductionist approach. Essentially, by acquiring more data, we can actually get away with analyzing fewer distinct conditions, by breaking complicated systems-level model inference questions into simpler sub-problems. To illustrate this reoccurring theme, consider the inference of metabolic regulation, the major topic of \hyperref[ch:simmer]{Chapter \ref{ch:simmer}}. If we are interested in identifying which of 50 metabolites ($m$) regulates each of 50 reactions of interest ($n$), then 2500, $\mathcal{O}(nm)$, possible metabolite-reaction pairs exist. This scale of inference is highly tractable, however, using existing approaches metabolic regulation is primarily tested at a systems-level (i.e. as a combined model-fit over all 50 reactions) \cite{Link:2013dj, Zampar:2013fr}.  In this case, the number of sets of possible regulatory relationships that could exist is 8.88 $\times 10^{84}$, $\mathcal{O}(n^{m})$, similar to the number of atoms in the observable universe. To make this problem scalable, I was able to achieve $\mathcal{O}(nm)$ complexity by breaking the dependence between reactions through measuring metabolic fluxes, rather than treating them as latent variables.

\subsection{Towards understanding how metabolism is controlled}

To allow for the application of piecewise inference, I focused on studying the baker's yeast, \textit{S. cerevisiae}, as a simple eukaryote that can be easily cultured and is amenable to genetic manipulation. Previous research in the Rabinowitz and Botstein groups has shown that when yeast are cultured in chemostats with greatly varying nutrient availability, both the transcriptome and metabolome are radically affected, but in a distinct manner \cite{Brauer:2008jn, Boer:2010fb}. Across these cultures, differences in growth rate and which nutrient is limiting growth (such as a carbon or nitrogen source), yields a panel of diverse physiological and metabolic states. As growth rates change, the transcriptome is strongly affected, with many transcripts increasing or decreasing in a log-linear fashion with growth rate, irrespective of which nutrient limits growth \cite{Brauer:2008jn}. In contrast, the metabolome is profoundly affected by which nutrient limits growth, with distinct metabolites appearing to be growth-limiting or overflowing depending on which primary nutrient limits growth \cite{Boer:2010fb}. While these studies convey the importance of both transcription and metabolism in nutrient-dependent growth control, each dataset is only able to provide a 1-dimensional answer to a question that inherently involves multiple `omic dimensions.

When yeast are growing at different rates and in media of greatly differing composition, they face a massive metabolic challenge. When growing quickly, metabolic rates (fluxes) must increase to produce enough  proteins, RNAs and other biomass components to keep pace with dilution.  Moreover, metabolic pathways must generally maintain an appropriate balance of energy, reducing equivalents and biomass components, across a broad spectrum of possible environments and genetic states. To achieve such fine control, pathway fluxes are collectively determined by enzyme expression, nutrient availability and metabolic regulation.  While analysis at this scale has been used to investigate the coordinated regulation of specific metabolic events involving small sets of genes \cite{Zampar:2013fr, Link:2013dj} as previously discussed this approach is ill suited to identifying novel regulation at scale. Instead, pathway-level regulation can be considered as an emergent property of interactions between reactions, where reaction-level kinetics are sufficient to reconstruct higher-order behavior \cite{Fell:1997wg}.

To characterize individual reactions, we can assess whether a model of reaction kinetics, such as Michaelis-Menten kinetics (\hyperref[intoEqtn:mm]{Equation \ref{intoEqtn:mm}}), is able to accurately predict the model output (flux) as a function of model inputs (metabolite and enzyme concentrations) and kinetic parameters \cite{Anonymous:1913wn, Liebermeister:2006fm, Tummler:2014cp}. Here, $\nu$ is the reaction flux, $\left[E\right]$ is the enzyme's concentration, k$_{cat}$ is the per-enzyme maximum rate of conversion of substrates into products ($\sfrac{1}{s}$), $\left[S\right]$ is the substrate's concentration, and k$_{m}$ is a measure the enzyme-substrate complex's reversibility.

\begin{equation}
\nu = k_{cat}\left[E\right]\frac{\left[S\right]}{\left[S\right] + k_{m}}\label{intoEqtn:mm}
\end{equation}

Under this simplified kinetic model, variable flux could be accomplished through two simple regulatory paradigms: either the concentration of enzymes changes proportionally to their flux, or the fractional substrate occupancy of each enzyme increases. For this second hypothesis, the upstream accumulation of metabolites could be a driving force that is directly controlled by nutrient availability. A combination of these strategies is also possible, where different enzymes could employ one or both of these methods of flux-control. 

Previous work from our laboratory, using transcript levels as an approximation of protein levels, has revealed that neither of these simple paradigms is able to fit the inferred distribution of fluxes \cite{Bradley:2009fj}. There is no consistent relationship between fluxes and transcript abundance for most enzymes, nor do metabolites accumulate upstream of enzymes as flux increases. This suggests that either transcript abundance is an unsuitable surrogate for enzyme abundance (a hypothesis that can be tested using protemics) or that we need to expand the model beyond simple Michaelis-Menten kinetics to account for other mechanisms of flux regulation, such as post-translational modifications, allostery and localization. Across different nutritional conditions, we must consider not just the differences in flux associated with growth rate, but also potential differences in cellular composition and energetics, such as tradeoffs between fermentation and respiration \cite{Lange:2001th, Feist:2010hq, BARFORD:1979ei}. These trends will result in differences in the relative fluxes through different pathways.

\subsection{Integrative `omic approaches to identify metabolic and post-transcriptional regulation}

To utilize this system to investigate metabolic regulation, I filled in two of the major experimental gaps in this dataset by measuring protein abundance across the 25 chemostat conditions using quantitative proteomics and by determining metabolism-wide fluxes using flux balance analysis (\hyperref[introFig:primarytopics]{Figure \ref{introFig:primarytopics}A}) \cite{Orth:2010hb}. As mass spectrometry data, in the form of proteomics and metabolomics data, is of particular importance in this project, during \hyperref[ch:quant_anal]{Chapter \ref{ch:quant_anal}} I discuss the statistical analysis of mass spectrometry data. 

During \hyperref[ch:simmer]{Chapter \ref{ch:simmer}}, I discuss the integration of reaction flux with enzyme and metabolite concentrations to test alternative models of reaction-level kinetics. The first aim of this study is determine whether Michaelis-Menten kinetics (\hyperref[intoEqtn:mm]{Equation \ref{intoEqtn:mm}}) is sufficient to quantitatively related physiological concentrations of substrates and enzymes to variable reaction flux.  To the extent that such an agreement is not possible, we then assessed whether metabolic regulation through allostery was able meaningful improve fit (\hyperref[introFig:primarytopics]{Figure \ref{introFig:primarytopics}B}).  This approach, called SIMMER (\underline{S}ystematic \underline{I}nference of \underline{M}eaningful \underline{M}etabolic \underline{E}nzyme \underline{R}egulation) is novel in that it is able to to scalably identify metabolic regulation. Accordingly, this approach was used to both confirm the physiological important of canonical yeast metabolic regulation and to identify novel regulation which was subsequently experimentally validated. By inferring reaction kinetics based on physiological variation in reaction species, SIMMER also naturally indicates the relative influence of substrates, products, enzymes and regulators in changing reaction flux. This motivates a simple heuristic approach based on \textit{metabolic leverage} which suggests which reactions are major control points in a manner that is consistent with, but less data intensive than metabolic control analysis \cite{Fell:1997wg}.

By deciding to directly measure protein abundance across the chemostat conditions, rather than relying upon transcript abundance as a surrogate for protein abundance I was guarding against possible deviations between protein and transcript abundance. Once protein relative abundance was measured, I was able to compare transcriptional changes to corresponding changes in protein abundance to determine whether the addition of proteomics was necessary (\hyperref[introFig:primarytopics]{Figure \ref{introFig:primarytopics}C}). The results of this analysis are presented in \hyperref[ch:pt_compare]{Chapter \ref{ch:pt_compare}}.  While strong transcriptional changes do indeed propagate into a corresponding change in protein abundance, additional regulation appears to be operating at the post-transcriptional level to further attune transcriptional responses to the metabolic environment. Regulation of protein degradation appears to be particularly important during nitrogen-limitation, where ribosomes are preferentially degraded and metabolic enzymes are selectively retained.

Code supporting each chapter is maintained on \href{https://github.com/shackett}{github}, access to private repositories is available upon request.

\begin{figure}[h!]
\floatbox[{\capbeside\thisfloatsetup{capbesideposition={right,top},capbesidewidth=4cm}}]{figure}[\FBwidth]
{\caption[Summary of primary thesis topics]{\fixedspaceword{\\Summary of primary thesis topics. \textbf{A)} A multi-omic data was generated involving four distinct classes of high-dimensional data. Previously measured transcript and metabolite relative abundances \cite{Brauer:2008jn, Boer:2010fb} were supplemented with measurements of enzyme relative abundances and inference of metabolism-wide fluxes. Each data type was measured across 25 standard chemostat conditions. \textbf{B)} Integration of metabolite, enzymes and flux measurements allows testing of reaction-level models of kinetics. Each possible model of kinetics is described as a non-linear function of substrates, products, enzymes, possible regulators and constants. Alternative kinetic models are compared based on model plausibility, $Pr(Model)$, and how well measured fluxes agree with the Michaelis-Menten kinetics, $Pr(Data | Model)$. \textbf{C)} The relationship between changes in transcript abundance and protein abundance can be modeled using differential equations. Departures between protein and transcript changes may reflect additional post-transcriptional regulated due to variable translational efficiency or protein degradation.
}}\label{introFig:primarytopics}}
{\includegraphics[width=\textwidth-4cm]{ch-intro/Figures/globalQuant.pdf}}
\end{figure}

