
\chapter{Introduction\label{ch:intro}}

As an undergraduate genetics major, I was fascinated with complex traits such as height, corn yield and heart attack risk. While these traits are of huge societal importance, each is collectively determined by numerous genetic variants and environmental factors; consequently, relating quantitative traits to their genetic determinants has been notoriously difficult \cite{Manolio:2009jp}. When I entered graduate school, I wanted to better understand the source of missing heritability and how the explanatory insufficiency of the GWAS paradigm could be addressed. My insight into this problem is that the gulf separating genetic variation and whole-body complex traits is the massive complexity of organismal and cellular physiology.

At the scale of cellular and organism-level systems, thousands of genes and environmental effects intersect; this leads to non-additive, non-linear interactions, such as those resulting in canalization \cite{Waddington:1942wy}. Because the behavior of cells across diverse physio-genetic states cannot be explained using additive genetics, the combination of many such subsystems into organism-level phenotypes would only further distort the relationship between individual genetic variants and overall phenotypic variability. Outside of tightly controlled studies of microbial growth \cite{Bloom:2013bq}, it is likely that additive genetics could only explain a minority of many phenotype's variation regardless of statistical power \cite{Weedon:2008gc,WellcomeTrustCaseControlConsortium:2007do}. Furthermore, genome-scale methods that aim to account for such non-additive and non-linear interactions in an unsupervised manner would be limited due to the intractable complexity and breadth of such interactions. An alternative to this unsupervised approach can be found in systems biology, where researchers continually build bottom-up models of systems operations based on a relatively small number of inputs.

\subsection{Harnessing the power of systems biology}

A single systems biology model may focus on a minute aspect of an organism. For example, one model may aim to describe how the expression of your favorite gene (YFG), is determined based on the activity of its primary transcriptional repressor and cis-regulatory variation in YFG \cite{Nuzhdin:2012ii}. Such a simple model of YFG expression could be used to characterize YFG protein abundance \cite{Jovanovic:2015hp}, which can in turn be built into a model of signaling or metabolic pathway activity \cite{Neves:2002bk, Chassagnole:2002ty}.  Each model importantly integrates information within the appropriate context of the systems operation, thus allowing changes in system output (and uncertainty) to be expressed as a non-linear function of variation in inputs.

In an ideal world where systems were well understood, the impacts of individual genetic variants and environmental effects could be propagated through individual subsystems; aggregating information to accurately predict quantitative traits and disease risk.  At such a juncture, prediction would require far less data than would be necessary to initially parameterize the system. Unfortunately, to date, creating systems level models which could be used for such purposes, has remained difficult, due to both extensive experimental requirements and general challenges in model identification.

The need for extensive experimentation and difficulty of model discovery lay at odds when designing experiments. Given fixed resources, there are tradeoffs between producing an experiment that measures all desired variables and characterizing a sufficient number of conditions (and replicates) to distill anything meaningful from the quantitative information. Moreover, this material limitation is exacerbated through the further distribution of funding among groups characterizing different systems, in different organisms, with limited direct cross-domain translatability. 

To simplify the complex model inference problems of systems biology, I employed an experimentally-guided reductionist approach. Essentially, by acquiring more data, we can actually analyze fewer distinct conditions. This can be achieved by parsing complicated, systems-level model inference questions into simpler subproblems. To illustrate this recurring theme, we can consider the inference of metabolic regulation, the major topic of \hyperref[ch:simmer]{Chapter \ref{ch:simmer}}. If we are interested in identifying which of 50 metabolites ($m$) regulates each of 50 reactions of interest ($n$), then 2500, $\mathcal{O}(nm)$, possible metabolite-reaction pairs exist.  Although this scale of inference is highly tractable, existing approaches primarily test regulation at a systems level (i.e. as a combined model-fit over all 50 reactions) \cite{Link:2013dj, Zampar:2013fr}.  In this case, the number of sets of possible regulatory relationships that could exist is 8.88 $\times 10^{84}$, $\mathcal{O}(n^{m})$, roughly the number of atoms in the observable universe. To make this problem scalable, I achieved $\mathcal{O}(nm)$ complexity by breaking the dependence between reactions. I did so by measuring metabolic fluxes, rather than treating them as latent variables.

\subsection{Towards understanding metabolic control}

To allow for the application of piecewise inference, I studied Baker's yeast, \textit{S. cerevisiae}; a simple eukaryote that can be easily cultured and is amenable to genetic manipulation. Previous research in the Rabinowitz and Botstein groups found that when yeast were cultured in chemostats with greatly varying nutrient availability, both the transcriptome and metabolome were radically affected, but in a distinct manner \cite{Brauer:2008jn, Boer:2010fb}. Across these cultures, differences in growth rate and growth-limiting nutrient (such as a carbon or nitrogen source), yield a panel of diverse physiological and metabolic states. As growth rates change, the transcriptome is strongly affected, with many transcripts increasing or decreasing in a log-linear fashion with growth rate, irrespective of which nutrient limits growth \cite{Brauer:2008jn}. In contrast, the metabolome is profoundly affected by which nutrient limits growth, with distinct metabolites appearing to be growth-limiting or overflowing depending on which primary nutrient limits growth \cite{Boer:2010fb}. While these studies convey the importance of both transcription and metabolism in nutrient-dependent growth control, each dataset is only able to provide a one-dimensional answer to a question that inherently involves multiple `omic dimensions.

When yeast are grow at varying rates and in media of diverse composition, they face a massive metabolic challenge. In an environment where yeast grow rapidly, metabolic rates (fluxes) must increase to produce enough  proteins, RNAs and other biomass components to keep pace with dilution.  Moreover, metabolic pathways must generally maintain an appropriate balance of energy, reducing equivalents and biomass components across a broad spectrum of possible environments and genetic states. To achieve such fine control, pathway fluxes are collectively determined by enzyme expression, nutrient availability and metabolic regulation.  While analysis at this scale has been used to investigate the coordinated regulation of specific metabolic events involving small sets of genes \cite{Zampar:2013fr, Link:2013dj} as previously discussed, this approach is ill suited to identifying novel regulation and reaction kinetics at scale. Instead, pathway-level regulation can be considered as an emergent property of interactions between reactions, whereby reaction-level kinetics are sufficient to reconstruct higher-order behavior \cite{Fell:1997wg}.

To characterize individual reactions, we can assess whether a model of reaction kinetics, such as Michaelis-Menten kinetics (\hyperref[intoEqtn:mm]{Equation \ref{intoEqtn:mm}}), accurately predicts the model output (flux) as a function of model inputs (metabolite and enzyme concentrations) and kinetic parameters \cite{Anonymous:1913wn, Liebermeister:2006fm, Tummler:2014cp}. Here, $\nu$ is the reaction flux, $\left[E\right]$ is the enzyme concentration, k$_{cat}$ is the per-enzyme maximum rate of conversion of substrates into products ($\sfrac{1}{s}$), $\left[S\right]$ is the substrate concentration, and k$_{m}$ is effectively the concentration where half of the enzymes are bound by substrates.

\begin{equation}
\nu = k_{cat}\left[E\right]\frac{\left[S\right]}{\left[S\right] + k_{m}}\label{intoEqtn:mm}
\end{equation}

Under this simplified kinetic model, variable flux can be accomplished through two simple regulatory paradigms: either the concentration of enzymes changes proportionally to flux, or the fractional substrate occupancy of each enzyme increases. For the latter hypothesis, the upstream accumulation of metabolites can be a driving force that is directly controlled by nutrient availability. A combination of these strategies is also possible, wherein different enzymes could employ one or both of these methods of flux-control. 

Using transcript levels as an approximation of protein levels, previous work from our laboratory has revealed that neither of these simple paradigms is generally consistent with inferred reaction fluxes \cite{Bradley:2009fj}. There is no consistent relationship between fluxes and transcript abundance for most enzymes, nor is there a reliable accumulation of substrates and flux increases. This suggests that either transcript abundance is an unsuitable surrogate for enzyme abundance (a hypothesis that can be tested using proteomics) or that we need to expand the model beyond simple Michaelis-Menten kinetics to account for other regulation, such as through post-translational modification, allostery or localization. Across different nutritional conditions, we must consider not only the differences in flux associated with growth rate, but also potential differences in flux due to variable cellular composition and energetics, such as tradeoffs between fermentation and respiration \cite{Lange:2001th, Feist:2010hq, BARFORD:1979ei}.

\subsection{Integrative `omic approaches for identifying metabolic and post-transcriptional regulation}

While analysis of transcriptional and metabolomic variation initially suggested that existing yeast chemostat conditions would be an ideal system to investigate the regulatory underpinnings of nutrient-based growth control, additional data was required to meaningfully address this question. In order to significantly reduce the gaps in our knowledge, I measured protein abundance across the 25 chemostat conditions using quantitative proteomics. I additionally estimated metabolism-wide fluxes using flux balance analysis (\hyperref[introFig:primarytopics]{Figure \ref{introFig:primarytopics}A}) \cite{Orth:2010hb}. Because metabolite and enzyme concentrations are the primary determinants of reaction flux, the appropriate quantification of these species via mass spectrometry is a major focus of my research. In \hyperref[ch:quant_anal]{Chapter \ref{ch:quant_anal}}, I discuss appropriate methods for analyzing mass spectrometry data and highlight collaborations where metabolomics or proteomics provided novel insights into physiology and evolution. 

Throughout \hyperref[ch:simmer]{Chapter \ref{ch:simmer}}, I discuss the integration of reaction flux with enzyme and metabolite concentrations to test alternative models of reaction-level kinetics. The first aim of this study is to determine whether generalizations of Michaelis-Menten kinetics (\hyperref[intoEqtn:mm]{Equation \ref{intoEqtn:mm}}) \cite{Liebermeister:2006fm} are sufficient to quantitatively relate physiological concentrations of substrates and enzymes to variable reaction flux.  To the extent that such an alignment is not possible, I assessed whether metabolic regulation through allostery is able meaningful improve fit (\hyperref[introFig:primarytopics]{Figure \ref{introFig:primarytopics}B}).  This approach, SIMMER (\underline{S}ystematic \underline{I}nference of \underline{M}eaningful \underline{M}etabolic \underline{E}nzyme \underline{R}egulation), is novel in that it scalably identifies metabolic regulation. Accordingly, this strategy was used to both confirm the physiological importance of canonical yeast metabolic regulation and to identify novel regulation that was subsequently experimentally validated. By inferring reaction kinetics based on physiological variation in reaction species, SIMMER also naturally indicates the relative influence of substrates, products, enzymes and regulators in changing reaction flux. This suggests that metabolic activity is largely determined by changes in substrate and product concentrations based on local metabolic context; direct regulation through changes in protein abundance and allostery primarily fine-tune metabolic behavior.

By directly measuring protein abundance across the chemostat conditions, rather than relying on transcript abundance as a surrogate for protein abundance, I guarded against possible deviations between protein and transcript abundance. Once protein relative abundance was measured, I compared transcriptional changes to corresponding changes in protein abundance to assess whether the addition of proteomics was necessary (\hyperref[introFig:primarytopics]{Figure \ref{introFig:primarytopics}C}). The results of this analysis are presented in \hyperref[ch:pt_compare]{Chapter \ref{ch:pt_compare}}.  While strong transcriptional changes  indeed propagate into a corresponding change in protein abundance, at the post-transcriptional level, additional regulation attunes transcriptional responses to the metabolic environment. Post-transcriptional regulation appears to be particularly important during nitrogen-limitation, during which ribosomes are preferentially retained despite transcriptional down-regulation.

Code supporting each chapter is maintained on \href{https://github.com/shackett}{Github}; access to private repositories is available upon request.

\begin{figure}[h!]
\floatbox[{\capbeside\thisfloatsetup{capbesideposition={right,top},capbesidewidth=4cm}}]{figure}[\FBwidth]
{\caption[Summary of primary thesis topics]{\fixedspaceword{\\Summary of primary thesis topics. \textbf{A)} A multi-omic data was generated involving four distinct classes of high-dimensional data. Previously measured transcript and metabolite relative abundances \cite{Brauer:2008jn, Boer:2010fb} were supplemented with measurements of enzyme relative abundances and inference of metabolism-wide fluxes. Each data type was measured across 25 standard chemostat conditions. \textbf{B)} Integration of metabolite, enzyme and flux measurements enables testing of reaction-level models of kinetics. Each possible model of kinetics is described as a non-linear function of substrates, products, enzymes, possible regulators and constants. Alternative kinetic models are compared based on model plausibility, $Pr(Model)$, and how well measured fluxes agree with the Michaelis-Menten kinetics, $Pr(Data | Model)$. \textbf{C)} The relationship between changes in transcript abundance and protein abundance can be modeled using differential equations. Departures between protein and transcript changes may reflect additional post-transcriptional regulation due to variable translational efficiency or protein degradation.
}}\label{introFig:primarytopics}}
{\includegraphics[width=\textwidth-4cm]{ch-intro/Figures/globalQuant.pdf}}
\end{figure}

