
\chapter{An integrative `omics approach to quantitative analysis of physiological metabolic flux control\label{ch:slimer}}

Enzyme and metabolite quantitation has been revolutionized by advances in mass spectrometry, but methods to integrate the resulting data remain needed. Here we present a quantitative, scalable strategy for revealing steady-state metabolic regulation by integration of metabolomic, proteomic, and flux measurements. It involves analyzing, on a reaction-by-reaction basis, whether fluxes across conditions are accounted for by a Michaelis-Menten relationship between enzyme, substrate, and potential regulator concentrations. We collected the required data for yeast growing at 25 different nutrients-limited steady states and applied this strategy to reveal the primary physiological flux control mechanisms for over 40 metabolic reactions, encompassing 32 instances of physiologically-relevant allosteric regulation. The identified regulation included classical feedbacks and unexpected cross-pathway connections. Quantitatively, half of flux control resided in substrate and one quarter in enzyme concentrations. For reversible reactions, the remainder resided largely in product levels, and for irreversible reactions in allosteric effectors. Thus, metabolic activity is substantially self-regulated by metabolites themselves.


\section{Introduction}

A crowning achievement of twentieth century biochemistry was determining the pathways by which organisms convert diverse nutrients into macromolecules \cite{Caspi:2014je}. Despite extensive knowledge of metabolic reaction networks, even in the best-studied model microbes, the means by which metabolic reaction rates (fluxes) are controlled remains incompletely understood. Specifically, while mechanisms controlling the expression and activity of many metabolic enzymes are known, how these multiple modes of regulation come together in cells to control reaction rates is largely unanswered.

Most metabolic regulatory mechanisms were discovered by isolating enzymes and studying their kinetics \textit{in vitro}. While powerful for proving specific regulatory interactions, this reductionist approach is less effective at revealing which mechanisms play a predominant role in cells. For example, the true impact of certain regulators may be evident only in the presence of physiologic concentrations of substrates and other metabolites \cite{Fell:1997wg, Tummler:2014cp}. Moreover, certain regulators may change only slightly in concentration across biological conditions and therefore matter relatively little, whereas others change greatly and thus dominate flux control \cite{Kacser:1973fe}. 

To rigorously understand control of metabolic fluxes in cells, quantitative analysis is critical. The framework of metabolic control analysis (MCA) uses flux control coefficients (C$^{J}_{E}$) to capture the importance of a given enzyme's concentration ([E]) to overall pathway flux (J), with C$^{J}_{E}$  = $(dJ/J)/(d[E]/[E])$ \cite{Kacser:1973fe}. Experimental determination of flux control coefficients is, however, difficult, and regulatory interactions in cells may result in flux being weakly linked to pathway enzymes \cite{Hauf:2000vu, Kochanowski:2013fb, Fell:1997wg, CornishBowden:1995fy}. For example, the rate of glycolytic flux may be determined by total cellular ATP demand rather than glycolytic enzyme expression \cite{Koebmann:2002ic}. Thus, there is a need for additional approaches to link available experimental data to flux control.

To this end, there has been growing interest in building differential equation models of metabolic dynamics. Each reaction is typically associated with multiple parameters (e.g., k$_{cat}$, k$_{m}$, k$_{i}$), with changes to the parameters describing any one reaction altering the entire system dynamics. By tuning enzyme kinetic parameters and sifting through possible regulators, such models can be fit to experimental metabolite concentration data \cite{Teusink:2000kc, Chassagnole:2002ty, Tummler:2014cp}. The difficulty is that parameter and regulator identification requires global non-linear search in high dimensional space. Accordingly, parameter and regulator identification scales poorly with network size and missing prior knowledge. Therefore, there is a need for new approaches that scale to the genome-level and are adaptable to systems with limited prior knowledge.

Here we present an approach that we term \underline{S}ystems \underline{L}evel \underline{I}dentification of \underline{M}etabolic \underline{E}nzyme \underline{R}egulation (SLIMER). By combining steady-state proteomic, metabolomic, and fluxomic data, SLIMER evaluates on a reaction-by-reaction basis the physiological mechanisms underlying flux control in a quantitative manner. Specifically, given measurements of fluxes, metabolites, and enzymes across multiple steady states, SLIMER tests, one reaction at a time, whether the observed fluxes can be explained by a Michaelis-Menten relationship between substrate, product, and enzyme concentrations (Figure 1A). When a misfit is observed, it then searches for potential regulators that significantly rectify the discrepancy (Figure 1B). Because parameters and regulators are identified one reaction at a time, the approach is scalable. 

Here we report integrative `omic analysis of 25 different steady-state yeast growth conditions, and subsequent application of SLIMER to reveal metabolic regulation. SLIMER recapitulates much of known regulation and reveals novel regulation. In addition, it determines the quantitative fraction of regulation residing in substrate/product concentration, enzyme concentration, and allosteric effector concentration. Strikingly, substrate/product concentration account for almost all flux control of reversible reactions, whereas enzyme and effectors also contribute significantly for irreversible ones. This provides unbiased quantitative support for the general paradigm of overt metabolic regulation being restricted to irreversible pathway steps. 

\section{Results}

\subsection{Data collection.}

We determined fluxes, metabolite concentrations, and enzyme concentrations, across 25 different steady-state yeast cultures. Yeast were grown in chemostats limited for carbon, nitrogen, phosphorus, leucine, or uracil, each at five different growth rates. Carbon, nitrogen, and phosphorous limitation were implemented in prototrophic yeast through manipulation of the media glucose, ammonium, and phosphate, respectively; leucine and uracil limitation were implemented in auxotrophs. Collectively, these conditions span a broad range of yeast physiology and metabolism.

To determine fluxes across these conditions, we employed flux balance analysis constrained by experimental measurements of nutrient uptake, waste excretion, and biomass generation. Uptake of glucose (the sole carbon source) and excretion of waste products were measured by $^{1}$H-NMR (Figure \ref{fig:boundFlux}). In addition, yeast composition was evaluated by measuring the fraction of protein, carbohydrate, RNA, DNA, polyphosphate, and fatty acids. Intriguingly, we observed notable differences in composition across conditions. This included, as expected based on prior literature \cite{Schulze:1995uv, Lange:2001th}, a strong positive correlation, irrespective of the limiting nutrient, between nucleic acid content (primarily in the form of ribosomal RNA) and growth rates, and a similar but less strong trend for total protein content. It also included differences across nutrient conditions, which were particularly evident at slow growth rate, including fat and polyphosphate accumulation in nitrogen limitation. Direct experimental determination of these differences in biomass was important for accurate flux determination, as the product of growth rate and fractional composition determines net biosynthetic fluxes (Figure \ref{fig:boundFlux}).

The combined uptake, excretion, and biomass production fluxes effectively constrained intracellular fluxes, whose values were determined by flux balance analysis by optimizing fit to experimental data and minimizing total network flux \cite{Herrgard:2008gb, Yizhak:2010jk}. The range of fluxes that were equally compatible with these constraints was determined by flux variability analysis (see methods) \cite{Mahadevan:2003wq}. This approach allowed us to determine the fluxes of 233 metabolic reactions (Figure \ref{fig:fluxHM}). With some important exceptions, such as more amino acid biosynthesis in leucine limitation and less glycolytic flux in glucose limitation, flux correlated strongly with growth rate.

With respect to metabolites, the relative concentrations of 106 metabolites were previously measured across these chemostat conditions by LC-MS/MS \cite{Boer:2010fb}. Overall, metabolite abundances depended strongly on the nature of the limiting nutrient, with marked depletion of key products, such as amino acids in nitrogen limitation and nucleotide triphosphates in phosphorous limitation. Here we augmented these relative measurements with absolute measurements, by an isotope ratio-based approach, for a single reference condition. From these data, we deduced absolute concentrations across all conditions. This enabled us to compare metabolite concentrations to biochemical measurements of enzyme site affinities.

An isotope ratio-based LC-MS/MS approach was also used to analyze the proteome. To determine relative protein abundances across conditions, each experimental sample was compared to a common $^{15}$N-labeled internal reference sample \cite{Oda:1999uz,Ong:2002tf}. Quantitative data was obtained for over 20,000 peptides representing 1187 proteins. Repeat measurements comparing slow-growing nitrogen and phosphorus-limited chemostats showed good reproducibility (Figure \ref{fig:proteomicsConsistency}). Unlike metabolites, the abundances of many proteins, especially ribosomal proteins, depended primarily on growth rate irrespective of the limiting nutrient (Figure \ref{fig:proteinHM}). Quantitatively, the fraction of concentration variation explained by growth rate alone was greater for the proteome than the metabolome, but less than for the transcriptome (Figure 2A) \cite{Brauer:2008jn}. 

The measured proteins included 370 metabolic enzymes, a number similar to that obtained by SRM-based approaches tailored for enzyme quantitation \cite{Costenoble:2011hia, Zampar:2013fr} (Table \ref{protTable}). Much like metabolite abundances, metabolic enzyme abundances varied strongly depending on the limiting nutrient. Compared to proteins overall, the relative concentrations of small molecule metabolic enzymes depended less on general growth rate and more on the specific limiting nutrient (Figure 2A). Notable nutrient-specific trends included down-regulation of glycolytic enzymes and up-regulation of TCA ones in carbon limitation, and increased levels of amino acid biosynthetic enzymes in leucine limitation (Figure 2B). Thus, relative to transcripts or proteins in general, the abundances of metabolites and metabolic enzymes are tied more strongly to the particular nutrient environment. In the case metabolites, this may directly reflect mass action, whereas for metabolic enzymes it may reflect regulation designed to maintain appropriate flux as nutrient and thus metabolite levels change.

\subsection{Analysis of triose-phosphate isomerase reaction with SLIMER.}

To illustrate the SLIMER methodology, consider the triose-phosphate isomerase (TPI; TPI1) reaction in glycolysis which converts dihydroxyacetone phosphate (DHAP) into glyceraldehyde 3-phosophate (GAP). Concentrations of substrate and product varied largely in tandem across the 25 chemostat conditions, with a trend towards increasing levels of both with faster growth (Figure 2A). Tpi1p concentration was lowest in carbon limitation and did not vary strongly with growth rate, even though flux increased markedly with faster growth (Figure 2A,B). Substrate, product, and enzyme concentrations can be related to flux through standard reversible Michaelis-Menten kinetics. For generality, a reversible rate law that assumes random-order enzyme mechanism and substrate/product competition was employed for all reactions \cite{Liebermeister:2006fm, Tummler:2014cp}. Kinetic parameters were identified by non-linear optimization to maximize the consistency of the Michaelis-Menten model and measured flux. For TPI, this approach, without invoking any overt regulation, was sufficient to explain most of the flux variability across the 25 conditions.

\subsection{Identifying physiological regulation.}

The same approach was used to study PRPP amidotransferase (PPAT; ADE4), the first committed step of purine biosynthesis.  During this reaction, an amino group is transferred onto PRPP from glutamine, liberating pyrophosphate.  Because this pyrophosphate is then broken down into organic phosphate by pyrophosphatase this reaction is effectively irreversible.  Neither of the substrates of this reaction responded strongly to growth rate, but both were strongly influenced by limiting nutrient; with PRPP specifically-depleted in phosphate limitation and glutamine in nitrogen limitation (Figure 3A).  Ade4p expression was lowest in carbon limitation and greatest in leucine limitation and expression weakly increased with growth rate, while flux increased strongly with growth rate reflecting elevated biosynthetic demands. Because Ade4p is roughly proportional to the average flux through each limiting nutrient, Michaelis-Menten kinetics is able to explain limitation-specific differences in flux, however levels of PRPP, glutamine and Ade4p are not able to jointly explain how flux increases with growth rate (Figure 3B).  To determine whether regulation could account for this inconsistency, we compiled a set of nine regulatory candidates of PPAT drawing on BRENDA's compendium of previously reported activators and inhibitors spanning all domains of life \cite{Scheer:2011df}.  For each regulatory candidate, PPAT's rate law was modified to appropriately reflect the regulation, and each rate law was evaluated to determine whether it could explain flux significantly better than the regulation-free rate law (q-value $<$ 0.1).  Of the nine regulatory candidates of PPAT tested, only inhibition by Adenosine monophosphate (AMP) meaningfully improved fit. At slow growth rates, lower biosynthetic requirements result in the accumulation of AMP which attenuates the excessive flux through PPAT (Figure 3B).  This feedback of AMP on the first committed step of purine biosynthesis is a highly logical control mechanism that has been demonstrated in numerous organisms \cite{WYNGAARDEN:1959wf, Jones:1982dn}.

While ADE4 is an unambiguous example of a single regulator improving the fit of Michaelis-Menten kinetics, evaluating the relative merits of supported regulation is complicated when multiple regulatory mechanisms improve fit.  In this case, because metabolites are strongly correlated across our 25 conditions, some tested regulators may improve fit solely by virtue of correlation to meaningful regulation.  Such regulation can not be discriminated based solely on how well a model fits the data, $Pr(Data | Model)$, but rather we must determine the relative support for alternative competing models, $Pr(Model | Data)$.  Using Bayes Theorem, the relative support for alternative models, $Pr(Model | Data)$, given the quantitative support for each model $Pr(Data | Model)$ can be found if we knew how inherently plausible each model was based on the literature, $Pr(Model)$ (Equation \ref{eqtn-bayes}) \cite{Gelman:2003vk}.

\begin{align}
Pr(Model | Data) Pr(Data) &= Pr(Data | Model) Pr(Model)\notag\\
Pr(Model | Data) &= Pr(Data | Model) Pr(Model) / Pr(Data)\notag\\
Pr(Model | Data) &\propto Pr(Data | Model) Pr(Model)\label{eqtn-bayes}
\end{align}

To inform our approach, we compiled a list of 729 measured regulators of 55 reactions collectively suggested based on 239 and 4038 citations in the yeast and non-yeast literature respectively \cite{Scheer:2011df}.  Among these numerous regulators, 22 regulators of 16 reactions can be distinguished as a \textit{de facto} gold-standard of yeast physiological regulation (Table \ref{tab:GS}) \cite{Jones:1982dn, Sekine:2007ej, Fraenkel:2011wp}.  To determine how likely it is that each regulator is physiological (i.e. $Pr(Model)$), we considered whether gold-standard yeast physiological regulation could be distinguished from primarily non-physiological non-gold-standard regulation by virtue of quantitative literature support from BRENDA.  Focusing on the 16 reactions with one or more gold-standard regulators, we found that the log-odds of a regulator falling in the physiological list increased strongly with both yeast (p $<$ 0.002) and non-yeast (p $<$ 0.0002) regulatory annotations and decreasing as the total number of regulators of a reaction assessed grew (p $<$ 0.02).

Using our regression-based model of of $Pr(model)$, the plausibility of a model of regulation by each regulator of both gold-standard and non-gold-standard reactions could be found. This prior knowledge naturally enforces both that most regulation is extremely unlikely \textit{a priori} (median probability of $\sim$ 0.03) and that for extremely well-studied reactions a larger proportion of supposed regulators are likely spurious.  Having appropriately balancing the likelihood, $Pr(Model | Data)$, and prior literature support $Pr(Model)$ for each possible regulatory model, alternative regulatory models can be compared on even-footing, yielding both examples where canonical regulation emerges, being reinforced by the prior, and other examples where strong quantitative support exists despite little literature backing.

In addition to the regulation of ADE4 by AMP, additional canonical regulators that were predicted over all alternative regulators include regulation of aspartokinase (HOM3), homocitrate synthase (LYS20/LYS21), glutamate-5-kinase (PRO1) and acetolactate synthase (ILV2,6) by threonine, lysine, proline and ATP, respectively \cite{WYNGAARDEN:1959wf, Jones:1982dn, Sekine:2007ej}. To verify, that our recall of well-supported regulation is not merely reflecting the prior, we can consider that regulation which is physiologically-supported by the literature should also tend to be kinetically important when investigated by our method.  Setting aside our inferred priors, 50\% of gold-standard regulators significantly improved fit compared to only 17\% of non-gold-standard BRENDA regulators (p $<$ 0.0005; Pearson's $\chi^{2}$ test). In these cases, the prior and empirical support reinforce each other, providing additional power to discriminate alternative regulatory hypotheses.

By reproducing the best supported regulation in yeast over weaker candidates, SLIMER distills \textit{in vivo} regulation when our results conform to prevailing knowledge, but the real power of this approach is that a multitude of understudied regulatory hypotheses can be efficiently evaluated and statistically compared.  To demonstrate the utility of this approach for computationally-guided discovery, we chose two well-studied reactions where predicted regulation departs from canon, and for each reaction, precise molecular hypotheses were tested using \textit{in vitro} biochemistry.  Studying ATP-phosphoribosyltransferase (ATP-PRTase; HIS1), we found that the canonical inhibition by histidine was likely insufficient, because this inhibition is strongly contingent upon ATP-PRTase's product, phosphoribosyl-ATP (Figure \ref{fig:atpprtase}).  This combinatorial regulation has previously been suggested in \textit{e. coli} but we are the first to report its relevance in \textit{S. cerevisiae} \cite{DallLarsen:1976wm}.  Investigating ornithine transcarbamylase (OTCase; ARG3), an enzyme that is not commonly thought to be regulated through allostery \cite{Jones:1982dn}, our method predicts that alanine could serve as a physiological inhibitor.  Using both spectrophotometric and mass spectrometric assays, we confirmed that alanine inhibits Arg3p with a k$_{i}$ of 15 mM (Figure \ref{fig:otcase}).  Given physiological concentrations, alanine, and to a lesser extent the other aliphatic amino acids, will differentially impact OTCase flux.  One implication of this regulation is that amino acid concentrations could indirectly stimulate ribosome synthesis, possibly allowing for a metabolic coordination of amino acid abundance and ribosome synthesis.

\subsection{Global analysis.}
 
By allowing researchers to evaluate the support for alternative kinetic models that can then be interrogated through directed experiments, SLIMER is a powerful tool to study individual reactions.  The greatest benefit of this approach however, is that the same dataset can provide comparable insight into reactions distributed across metabolism.  Our data were sufficient to inform the kinetics of 56 reactions across central carbon metabolism, amino acid synthesis and nucleotide metabolism.  For each reaction, reversible Michaelis-Menten kinetics was compared to models of regulation drawn from BRENDA and for those regulatory mechanisms that significantly improved fit, significant regulatory mechanisms were augmented by either allowing for regulator cooperativity or including a second regulator. The support for each kinetic model can be summarized based on how well it conforms to the data (Pr(data$|$model)) and its plausibility (Pr(model$|$literature)).  These models can be compared, while penalizing complicated models using Akaike Information Content with Correction (AICc) \cite{Burnham:2002eq}. Through model comparison, the best supported kinetic model of each reaction was determined, while we we also noted alternative supported regulatory mechanisms for each reaction (Table S2).

Inspecting the best-fit reaction forms for each of the 56 studied reactions, the kinetics of twelve reactions could not be approximated by any tested model (spearman correlation $<$ 0.6), while the remaining 44 reactions were reasonably fit by either generalized Michaelis-Menten kinetics (16 reactions) or models including some regulation (28 reactions) (Figure 4).  For eight of these reactions, further improvement was possible through the addition of a secondary regulator, or allowing for regulator cooperativity.  Metabolite affinities implied by the reaction forms agree with literature estimates \cite{Scheer:2011df}, with substrates and products varying between unsaturated and saturated on a case-by-case basis, but regulators operating at concentrations near their affinity, where the greatest tuning of flux is possible (Figure \ref{fig:occupancy}). Inferred thermodynamics are consistent with previous estimates of free energy drops across glycolysis \cite{Flamholz:2013io}, and for near-equilibrium reactions, there is an important role for metabolite concentrations shifting the ratio of forward to reverse flux across nutrient conditions (Figure \ref{fig:paramaterValues}).

While we are exploring a large number of possible regulators drawn from BRENDA, these candidates still represent a small fraction of possible regulatory interactions that could exist. Identifying purely novel regulation may require checking all metabolites, but the the challenge with performing such an unsupervised search, is that we have measured many metabolites and they are highly correlated.  To make this search tractable, for each reaction, an activator and inhibitor were found whose relative abundance was governed by the metabolomic principal components, and whose inclusion resulted in the greatest consistency between measured and predicted flux.  While this approach is not generally suited for discovery due to limited statistical interpretability and an inability to discriminate quantitatively similar metabolites, it can still serve as a method for suggesting possible regulators when either no tested regulation is quantitatively supported or few regulatory hypotheses exist (Figure \ref{fig:hypoMet}).  Using this approach, we see that reactions with no adequate reaction form could not be rescued by an optimal regulator, suggesting some greater pathology than missing regulation, while for understudied regulation, one reaction, guanylate kinase (GUK1) showed evidence of regulation.

\subsection{Assessing how metabolites and enzymes change flux.}

In order to reach a clean alignment of measured flux and a kinetic model's prediction, this model implicitly accounts for the extent of concentration variability for each reaction specie and the impact of this variation on flux.  Not all reaction species are equally important in this regard; some species may remain at nearly fixed concentration or changes in their concentration may not change flux (e.g. saturated metabolites or minor isoenzymes).  To understand which species are important in driving variable flux through each reaction, we need a quantitative way to dissect the combined role of all species in driving variable flux into the marginal contribution of each metabolite and enzyme.  A specie's role in driving variable flux is related to two factors: the magnitude by which a specie's concentration varies and the way in which flux through the reaction responds to changes in this specie's concentration.  The influence of a single specie, its \textit{metabolic leverage}, is roughly proportional to the product of its fold-change and average elasticity $\left(\epsilon = \frac{\partial F}{\partial S}\frac{[S]}{F}\right)$ \cite{Kacser:1973fe, Liao:1993in}.

Calculating metabolic leverage for each of the 44 well-fit reactions, allows us to determine how the total variable flux in these reactions has resulted from changes of reaction species.  Summarizing the relative influence of substrates, products, enzymes and regulators (Figure 5A-C, \ref{fig:MLbar}) - we see that reversible reactions are almost entirely driven by changes in substrate and product concentrations although the relative influence of substrates and products varies from reaction to reaction.  In contrast, irreversible reactions are driven primarily by substrate and enzyme concentrations, and for select reactions, regulators also have an important role. In the vast majority of cases, even when strong regulation exists, a single regulator rarely dominates, instead, flux is collectively dictated by substrates, enzymes and regulators.

When reactions respond strongly to changes in enzyme or regulator concentrations, it is likely that these regulatory events causally change flux or metabolite concentrations.  We can test this assertion, which links metabolic leverage to metabolic control, by studying glycolysis where reaction kinetics are well-fit, allowing a comparison of reaction-level and pathway-level inference.  Using the optimal reaction forms for glycolytic reactions, two metabolic control analysis (MCA) models of glycolysis were created which contrast the metabolic consequences of two similarly supported regulatory mechanisms for pyruvate kinase \cite{Cortassa:1994is, Westerhoff:1987jo}. This model reproduces the finding that glycolytic flux is predominately governed by phosphofructokinase (PFK; PFK1,2) \cite{Cortassa:1994is}, while the feed-forward regulation on pyruvate kinase (PyK; CDC19, PYK2) by fructose 1,6-bisphosphate primarily changes metabolite concentrations rather than flux per-se in line with experimental findings\cite{Xu:2012gg} (Figure \ref{fig:MCA}). Projecting metabolic leverage onto a metabolic map (Figure 5D), it is clear that PFK and PyK are the most important regulatory reactions in glycolysis.  This agreement suggests that metabolic leverage can provide a first-glance at the control of pathways when comprehensive analysis using MCA is not immediately possible (Figure \ref{fig:MLpathways}).  For instance, production of structural glucans is governed by the concentration of the beta-glucan synthases (FKS1, GSC2), while the production of aspartate family amino acids, as well as the interconversion of purines and pyrimidines operates without external regulation; instead, flux is driven solely by changes in substrate and product concentrations. 

\section{Discussion}

As we move from understanding the behavior of individual metabolic pathways to understanding metabolism as a complete system, there is a pressing need for methods that excel at these larger scales. Previous attempts to infer metabolic regulation have evaluated model support at a systems-level.  This has forced researchers either to use strong \textit{a priori} assumptions about how metabolism functions (and thereby only explore a small amount of hypothesis space) \cite{Chassagnole:2002ty, Zampar:2013fr} or to restrict their focus to relatively small metabolic networks that could be comprehensively interrogated \cite{Link:2013dj}, albeit with poor scaling ($O(n^{m})$ for $n$ reactions and $m$ possible reactions forms of each reaction. By measuring flux, rather than inferring it based on changing metabolite concentrations, reactions can be rendered independent, and metabolism can be studied on a reaction-by-reaction basis. When this approach is adopted, metabolic inference scales linearly with the number of reactions and the number of regulators we want to consider for each reaction ($O(nm)$). Furthermore, each reaction-level model can be evaluated more quickly than metabolism-level models, and correct models of individual reactions can be found without necessitating that the kinetics of all reactions be correct. In our study, we used this approach to push beyond pathway-level analysis to investigate metabolic regulation at the level of complete metabolism. We used this approach to address two outstanding challenges in chemical biology: how to systematically identify physiological regulation that may be entirely novel and whether `omic data can elucidate the physiological control of flux.

These topics are extremely important in microbial metabolic engineering, where researchers frequently attempt to preferentially direct flux through pathways of interest.  Because microbial metabolism has evolved chiefly to exhibit robustness, native regulation ingrained in the metabolic network is frequently at odds with design goals \cite{Kitano:2007cp}.  As such, pathway engineering efforts are frequently hampered when cryptic metabolic regulation exists or the relationship between pathway fluxes and enzyme abundance is unclear.  Using SLIMER, solutions to both challenges can be found.  When a native allosteric regulator is important, this regulation can be identified based on its physiological impact, allowing targeted experiments to abolish the regulators binding. To determine how enzyme levels should be tweaked to alter pathway flux, we hypothesize that for select reactions where flux strongly tracks enzyme abundance, by mimicking physiological control mechanisms, over-expressing enzymes will result in changes in flux.  Our analysis suggests that such examples are rare, paralleling experimental observations that pathway fluxes are rarely limited by the expression of a single enzyme \cite{Hauf:2000vu, CornishBowden:1995fy}.  For such pathways where flux dependents upon multiple reactions, we have demonstrated, using glycolysis as an example, that the physiological reaction forms which we have inferred can be coupled together to provide a more accurate characterization of pathway control.

In this study, we have focused on characterizing the metabolism of \textit{S. cerevisiae}, a microbe whose metabolism is relatively well understood, but our approach can be applied to any organism that can be grown in suspension where metabolic behavior can be varied using either nutritional or genetic perturbations.  We feel that this approach will be particularly useful for providing a preliminary understanding of lesser understood organisms, where traditional approaches based on studying one enzyme at a time, are intractable.  Beyond providing such a one-shot characterization of metabolism, most lesser understudied organisms will contain reactions which have been scarcely biochemically characterized.  We have provided solutions for identifying regulation using little prior literature knowledge by either capitalizing upon regulatory conservation or searching for purely novel regulation by an optimum regulator.

While our data is the most comprehensive enzyme-metabolite-flux dataset to date, it is still not a complete collection of all relevant metabolic information.  To fulfill the promise of SLIMER will require both progressively refining our input data and and expanding the breadth of our analysis.  Improving our current data will largely depend upon improving the number of distinct metabolites that can be measured through metabolomics and improving the accuracy of flux determination.  Here, advances in instrumentation paired with more focused analytical methods should afford both a more accurate and comprehensive detection of relevant metabolites, and the use of stable isotope labeling could further improve flux accuracy \cite{Yuan:2008er}.  Expanding the breadth of our dataset will entail addressing biological phenomena that were beyond the scope of this study.  Chiefly, we are unable to assess the extent to which flux is impacted by post-translational regulation \cite{Fiedler:2009hx, Schulz:2014eo}, variable sub-cellular localization \cite{Kitamoto:1988wc} or cell-to-cell variability \cite{Tu:2005cv}.  These limitations are likely to blame for the 12 reactions where no adequate kinetic model could be found, although inaccurate measurements or missing regulation could also result in inappropriate regulatory predictions that partially compensate for unmeasured factors.  A final issue of note is that we had to make assumptions about the form of rate laws, and while true \textit{in vivo} rate laws rarely reduce to generalized Michaelis-Menten kinetics \cite{Hill:1977vm}, our reaction forms are able to achieve an approximately correct relationship between species and resulting flux across the physiological conditions investigated and thus these simplified kinetics are functionally sufficient for our purposes \cite{Fell:1997wg}.

Because we have characterized a reproducible set of conditions that contain much of the meaningful metabolic variation in yeast, we hope that this study will serve as an important scaffold for future work to build upon, whether by incorporating additional experimental information on existing conditions or by expanding the breadth of the study by characterizing additional conditions. Regulation can only be identified if it differentially impacts study conditions. By expanding the number of conditions that are investigated, we can unmask regulation that is important in these conditions, and our approach can identify this regulation by virtue of the kinetic importance it gains in these new conditions. This will also serve to refine the accuracy of parameter estimates from reaction forms and better discriminate  competing regulatory hypotheses.



\begin{figure}
\caption[We can infer \textit{in vivo} reaction kinetics if we have measured the reaction's rate and concentrations of enzymes and all relevant substrates, products and effectors.] { Using `omic methods, these quantities can be measured in a high-throughput manner, allowing the kinetics of many reactions to be inferred in parallel.  \textbf{A)} For reactions that follow simple Michaelis-Menten \textit{in vivo} kinetics, measurements of enzyme and substrate/product concentration should agree with measured flux.  \textbf{B)} For reactions with physiologically-meaningful allostery, this regulation must be added before flux can be accurately predicted.  If multiple regulatory candidates exist, better supported candidates will result in greater improvement in flux prediction.  For reactions where none of the tested regulators results in an accurate prediction of flux, other unaccounted for regulation may be kinetically important.}
\end{figure}

\begin{figure}
\caption[Summary of input data.]{ \textbf{A)} The relative impact of growth rate and limiting nutrients in driving systematic variation of transcripts, proteins, enzymes and metabolites was tested through comparison of nested regression models.  Proceeding from simpler to more complicated models, the fraction of variation explained across all genes/metabolites explained by the added term indicates its relative impact on abundance. \textbf{B)} Heatmap summarizing the variable concentration of 370 enzymes across chemostat conditions. From the ratio of unlabelled sample peptides to matching $^{15}N$-labelled reference peptides, the abundance of each enzyme relative to a common reference was determined.  Across the 25 conditions, enzyme relative abundances were mean-centered, log-transformed and organized by hierarchical clustering using pearson correlation with average linkage. }
\end{figure}

\begin{figure}
\caption[Michaelis-Menten kinetics is sufficient to relate substrate, product and enzyme concentrations to flux carried for the reaction triose-phosphate isomerase.] {\textbf{A)} Across 25 conditions, we measured the relative concentration the reaction substrate (DHAP: dihydroxyacetone phosphate), product (GA3P: glyceraldehyde 3-phosphate) and enzyme (TPI: Tpi1).  \textbf{B)} The simple Michaelis-Menten reaction form translates the 25 sets of reaction species into predict flux ($V_{rMM}$) based on the value of four inferred kinetic parameters.  Comparing measured flux to the reaction form's prediction indicates that flux through TPI can be explained by a Michaelis-Menten relationship between substrate, product and enzyme.}
\end{figure}

\begin{figure}
\caption[Two models of PRPP Amidotransferase (PPAT) kinetics are compared: reversible Michaelis-Menten kinetics with and without allosteric inhibition by AMP.]{ \textbf{A)} Across 25 conditions, we measured the relative concentrations of both substrates (PRPP:phosphoribosyl pyrophosphate and Gln:glutamine), and one product (Glu:glutamate) and the enzyme (ADE4). Unmeasured species were treated as being invariant across the measured conditions. \textbf{B)} If Michaelis-Menten kinetics is approximately true \textit{in vivo}, then measured flux should be a Michaelis-Menten function of substrates, products, and enzymes concentrations.  While Michaelis-Menten kinetics predicts the average flux through chemostats with the same limiting nutrient, it is unable to account for increasing flux with growth-rate.  \textbf{C)} Using measured concentrations of AMP (adenosine monophosphate), treating AMP as allosteric inhibitor results in a statistically-significant improvement in how well flux can be predicted relative to the simple Michaelis-Menten kinetics (p $<$ 3x10$^{-5}$, q $<$ 0.1).}
\end{figure}

\begin{figure}
\caption[Metabolism-wide summary of the consistency between measured flux and Michaelis-Menten predictions.]{ For each of 56 reactions, the spearman correlation between measured flux and flux implied by Michaelis-Menten kinetics (with and without regulation) is shown.  For 29 reactions, inclusion of a regulator was supported, and for eight of these reactions, further improvement is possible through a secondary regulator or regulator cooperativity.  For reactions with supported regulation, the improvement in spearman correlation relative to the best-supported simpler models is shown.}
\end{figure}

\begin{figure}
\caption[Flux change in response to changing concentrations of metabolites and enzymes whose impact on flux is governed by the metabolites' and enzyme's elasticity.] {The impact of a single metabolite or enzyme in driving variable flux is roughly proportional to its elasticity weighted by the standard deviation of log relative concentrations across physiological conditions (phosphate, carbon and nitrogen limitations).  For a reaction, the relative contributions of its enzymes, substrates and regulators in driving variable flux across physiological conditions can be approximated by this metabolic leverage.  \textbf{A)} Metabolic leverage can be projected onto a ternary surface where each reaction is effected by enzymes, substrates, products, and possibly regulators whose influence sums to 100\% leverage.  \textbf{B)} This metric is able to distinguish reactions that are irreversible and thus likely to be good points of regulation from more passive reversible reactions. Reactions are classified as kinetically irreversible if literature suggests that net flux only proceeds in the forward direction \cite{Heavner:2012dg}. \textbf{C)} Pie chart summary of average metabolic leverage for irreversible and reversible reactions. \textbf{D)} Projecting metabolic leverage onto a metabolic network highlights the major regulaotry control points that our method is able to reproduce and discover.}
\end{figure}

\input{ch-slimer/rOCA_materialsAndMethods}

