
\chapter{Statistical Analysis of Mass Spectrometry Data\label{ch:quant_anal}}

Mass spectrometry is a powerful tool for detecting and quantifying chemically heterogenous species, particularly metabolites and protein-derived peptides. A central requirement for mass spectrometry is that a specie can ionize. This generally involves the addition or removal of a proton, (H$^{+}$), from a small molecule. Because whole proteins vary greatly in size, they are generally studied by first enzymatically digesting intact proteins and then measuring the resulting peptides. Species are distinguished due to the fact that when moving through a constant magnetic field, ions with different mass-to-charge ratios ($\sfrac{m}{z}$) will accelerate at characteristic rates, allowing discrimination by virtue of differential procession frequency about an orbitrap or by differences in time-of-flight \cite{Boyd:2011vt}. Having separated ions based on their $\sfrac{m}{z}$, their abundance can be determined based on the current induced in a detector.  Because multiple chemical species can have very similar (or identical) $\sfrac{m}{z}$, and thus cannot be resolved by mass spectrometry, further discrimination through liquid or gas chromatography is routine.  Using chromatography, species are separated according to how long they take to reach the detector (i.e. retention time; RT) because of differential affinity for the chromatography column/solvent. Quantitatively, a single LC-MS run effectively yields a matrix of $\sfrac{m}{z}$ $\times$ RT where values are the current of detected ions.  

In mass spectrometry, each ion corresponds to an abundance peak protruding from the $\sfrac{m}{z}$ $\times$ RT plane (\hyperref[ch-quant_anal:MSworkflow_p1]{Figure \ref{ch-quant_anal:MSworkflow_p1}}). In order to determine the identity of each ion, two primary approaches are used.  When studying metabolites, most species of interest, such as amino acids and metabolites in glycolysis, are shared between species and pure standards can be acquired.  By subjecting purified standards to LC- or GC-MS, the characteristic retention time of these species can be found, enabling the identification of ions in subsequent experiments based on shared retention time and exact mass of standards.  If obtaining pure standards of the species of interest is not feasible, as is the norm when peptides are measured, tandem-mass spectrometry can be used to yield structural information of unknown ions \cite{Nesvizhskii:2005jp, Huang:2012tl}.  Once a subset of the detected ions have been identified as metabolites or peptides, the abundances of these meaningful species can be compared to determine how experimental samples differ.

To illustrate the challenges and opportunities of mass spectrometry, in this chapter I will address the statistical nature of mass spectrometry data and the application of mass spectrometry to investigate challenging biological questions.  While mass spectrometry data can be approximately modeled with a log-normal distribution, there is an excess of ``outliers'' that could muddle statistical analysis if not properly treated \cite{Nesvizhskii:2005jp, Cox:2008ir}. During the first section of this chapter, I will discuss the magnitude of deviations from log-normality, and I will additionally address the use of variance modeling to appropriately describe relative measurement uncertainty without invoking the ``outlier'' label.  During the second section, I will briefly discuss four studies that I have coauthored. Each work harnesses proteomics or metabolomics to better understand native physiology or metabolism. These studies analyze mass spectrometry data using non-parametric approaches that provide robustness to outliers without requiring strong parametric assumptions.

\input{ch-quant_anal/MS_analysis}


\section{Analysis of Mass Spectrometry Data}

Because metabolites and proteins are the central drivers of metabolism and physiology, mass spectrometry data, when appropriately analyzed, can provide powerful insights into cellular function.  To illustrate this, I will discuss several published studies that I have coathored, focusing on my data analysis contributions. Partially owing to the analysis of human pancreatic tumor metabolites, Commisso et al. 2013 and Kamporst et al. 2015 highlight the discovery of a novel way by which tumors acquire nutrients \cite{Commisso:2013hz, Kamphorst:2015cc}. In Bruenig et al. 2014, the relationship between segregating genetic variation and metabolite concentrations is investigated, revealing both genetic variants that perturb local metabolite concentrations and those with far reaching effects on metabolite concentrations and flux \cite{Breunig:2014bu}.  Finally, in Mathew et al. 2014, proteomics is used to identify classes of proteins that are specifically degraded or spared during autophagy \cite{Mathew:2014hz}.

\subsection{Identifying a metabolomic signature of protein eating in pancreatic tumors}

As part of the Stand Up to Cancer (SU2C) \href{http://www.standup2cancer.org/dream_team_members#Pancreatic}{Pancreatic Cancer Dream Team} (a diverse coalition of clinical and laboratory researchers), the Rabinowitz lab was tasked with determining how pancreatic tumors metabolically differ from normal pancreatic tissue.  It is increasingly apparent that metabolic transformations, such as aerobic glycolysis (i.e. the Warburg effect), are a cardinal feature of cancer progression \cite{VanderHeiden:2009gq, Hanahan:2011gu}.  Most of these changes involve increased consumption of nutrients, along with commensurate production of waste products, such as lactate and reactive oxygen species.  While higher rates of nutrient utilization do not necessitate a corresponding change in metabolite concentrations, growth-limiting metabolites generally deplete; by contrast, ``overflow'' metabolites tend to accumulate, yielding patterns that could be detected through metabolomics \cite{Boer:2010fb}.

From 49 human pancreatic ductal adenocarcinoma (PDAC) patients, paired tumor and benign adjacent pancreatic tissue samples were obtained; the samples were frozen in liquid nitrogen following surgical resection.  A total of 189 samples were analyzed, including at least one sample from each tumor and benign specimen and a hodgepodge of technical and biological replicates.  Metabolites from each sample were analyzed using three LC-MS methods geared towards the quantification of partially overlapping sets of compounds. Across all three methods, 127 distinct metabolites were detected from a total of 188 features.  To both properly analyze this dataset and extract meaningful differences between tumor and benign tissue, we controlled for between-individual variation, accounted for differences in the total signal of mass spectrometry runs and integrated data from multiple measurements of a single metabolite into a consensus signal. The full quantitative analysis of this dataset is publicly available on \href{https://github.com/shackett/Pancreatic_tumor_metabolomics}{GitHub}.

\subsubsection{Data normalization and processing}

Ion counts were normalized to correct for differences in total metabolite abundances across samples and for any sample-to-sample drift in the overall instrument response factor.  A normalization factor ($\gamma_{i}$) was calculated for each LC-MS run \textit{i}.  To calculate $\gamma_{i}$, every known metabolite peak P$_{mi}$ in LC-MS run \textit{i} was quantified, and compared to the median value of peak \textit{m} across all samples, $\mu_{m}$.  The scaling factor was calculated according to \hyperref[ch-quant_anal:normal]{Equation \ref{ch-quant_anal:normal}}, and ion counts were then corrected by P$^{*}_{mi}$ = $\frac{P_{mi}}{\gamma_{i}}$.

\begin{equation}
\gamma_{i} = \text{median}\left(\frac{P_{mi}}{\mu_{m}}\right)
\label{ch-quant_anal:normal}
\end{equation}

The normalized ion count matrix was log$_{2}$ transformed and averaged over replicates.  The similarity of pairs of unnormalized and normalized technical replicates is shown in \hyperref[ch-quant_anal:replicate_corr]{Figure \ref{ch-quant_anal:replicate_corr}}.  Because the average signal strength was unimportant, metabolite relative abundances were centered with respect to the row mean.  When a metabolite was measured on multiple instruments, a single abundance value was calculated by taking the mean of non-missing observations across all instruments.

\begin{figure}[h!]
\begin{center}
\subfloat{
\includegraphics[width=0.47\textwidth]{ch-quant_anal/Figures/bioRepCorr_unnorm.pdf}
}
\hspace{1mm}
\subfloat{
\includegraphics[width=0.47\textwidth]{ch-quant_anal/Figures/bioRepCorr_norm.pdf}
}

\caption[Comparison of the log$_{2}$ abundances of unnormalized ion counts for pairs of technical replicates]{Comparison of the log$_{2}$ abundances of unnormalized ion counts across all pairs of technical replicates.  \textbf{A)} Before normalization. \textbf{B)} After normalization.}
\label{ch-quant_anal:replicate_corr}
\end{center}
\end{figure}

\subsubsection{Quantifying the significance of tumor:benign differences}

To determine whether shared similarities between tumor samples were driving overall metabolite abundances, all 49 tumor and benign samples were grouped using hierarchical clustering (\hyperref[ch-quant_anal:all_sample_heatmap]{Figure \ref{ch-quant_anal:all_sample_heatmap}}). From this analysis, it is clear that the grouping of samples is primarily driven by person-to-person differences in metabolite abundance.  This extensive biological variance should not be surprising; tumor and benign samples share strong metabolic effects related to diet, circulating metabolite levels and body composition \cite{Kastenmuller:2015bk}.  In addition, because each tumor and benign pair was analyzed on the same day, these samples may share technical variation.  We are primarily interested in how tumor metabolism specifically perturbs normal metabolite levels, so person-to-person variability and technical variation are largely nuisance variables. Both of these effects can be largely controlled for by analyzing the difference in metabolite relative abundance between paired tumor and benign samples. 

\begin{figure}[h!]
\begin{center}
\includegraphics[width=1\textwidth]{ch-quant_anal/Figures/total_heatmap_colun.pdf}
\caption[Heatmap assessment of the relative impact of tumor biology and person-to-person variability on metabolite abundances.]{Heatmap assessment of the relative impact of tumor biology and person-to-person variability on metabolite abundances.  Tumor and benign samples are indicated above the heatmap.  Rows and columns were organized through hierarchical clustering, using pearson correlation as a distance metric and using average linkage.}
\label{ch-quant_anal:all_sample_heatmap}
\end{center}
\end{figure}

To determine whether a subset of metabolites are systematically higher or lower in cancerous tissue than in benign adjacent pancreatic tissue, p-values were computed using a paired t-test, and the null distribution of these t-statistics was found using the bootstrap \cite{Efron:1986cv}.  This method was chosen over determining the test significance against a t-distribution because the parametric t-distribution approach makes the assumption that the normal and tumor log-abundances are each normally distributed.  This assumption is not valid for many metabolites and would result in anti-conservative hypothesis testing if errantly invoked \cite{Schmoyeri:1996uh}.

\subsubsection{A bootstrap approach for estimating significant differences between paired samples}

For a given metabolite \textit{m}, with \textit{n} non-missing values, there will be \textit{n} measurements of normal-tissue metabolite abundance (N$_{mi}$), paired with \textit{n} measurements of tumor abundances (T$_{mi}$) from the same patients.  The abundances [N$_{mi}$, T$_{mi}$] were jointly standardized so that they collectively have a mean of zero and a standard deviation of one.  

The systematic difference between pairs can be captured by a paired t-test statistic (\hyperref[ch-quant_anal:ttest]{Equation \ref{ch-quant_anal:ttest}}).

\begin{align}
t_{m} = \frac{\sum_{i = 1}^{n}\frac{(N_{mi} - T_{mi})}{n}}{\sqrt{\frac{\sum_{i = 1}^{n}\frac{(N_{mi} - T_{mi})^{2}}{n-1}}{n}}}\label{ch-quant_anal:ttest}
\end{align}

To generate samples for an empirical null distribution, we first need to generate data in which the systematic variation between the tumor and normal samples has been removed. Then, we use the remaining variation to determine how often such a large systematic difference between tumor and normal samples (t$_{m}$) would be expected by chance.  To generate this null data: the paired difference was removed from the normal abundances (\hyperref[ch-quant_anal:residvar-1]{Equation \ref{ch-quant_anal:residvar-1}}), the modified tumor and normal abundances were recentered (\hyperref[ch-quant_anal:residvar-2]{Equation \ref{ch-quant_anal:residvar-2}}), and finally these residuals were inflated in order to account for the 1 degree of freedom eliminated by removing the paired difference (\hyperref[ch-quant_anal:residvar-3]{Equation \ref{ch-quant_anal:residvar-3}}).

\begin{subequations}
\begin{align}
N^{*}_{mi} &= N_{mi} - \frac{\sum_{i = 1}^{n}(N_{mi} - T_{mi})}{n}\label{ch-quant_anal:residvar-1}\\
C_{m} &= \text{mean}[N^{*}_{mi}, T_{mi}]\label{ch-quant_anal:residvar-2}\\
N^{r}_{mi} &= (N^{*}_{mi} - C_{m})\sqrt{\frac{n}{n - 1}}\notag\\
T^{r}_{mi} &= (T_{mi} - C_{m})\sqrt{\frac{n}{n - 1}}\label{ch-quant_anal:residvar-3}
\end{align}
\end{subequations}


To generate each bootstrap sample (R = 500,000 in this study), a vector of indices of length n was sampled with replacement from [1...n]. These indices were used to sample pairs of residuals from N$^{r}_{mi}$ and T$^{r}_{mi}$: N$^{bs}_{mi}$ and T$^{bs}_{mi}$.  A bootstrapped null test statistic could then be calculated from \hyperref[ch-quant_anal:bs]{Equation \ref{ch-quant_anal:bs}}.

\begin{equation}
t^{bs}_{mr} = \frac{\sum_{i = 1}^{n}\frac{(N^{bs}_{mi} - T^{bs}_{mi})}{n}}{\sqrt{\frac{\sum_{i = 1}^{n}\frac{(N^{bs}_{mi} - T^{bs}_{mi})^{2}}{n-1}}{n}}}\label{ch-quant_anal:bs}
\end{equation}

This bootstrapped null distribution of statistics t$^{bs}_{mR}$ can be compared to the test statistic \textit{t}$_{m}$ using the percentile method to generate a p-value (\textit{p}$_{m}$) for each metabolite (\hyperref[ch-quant_anal:bspval]{Equation \ref{ch-quant_anal:bspval}}). Correction for multiple hypothesis testing followed the procedure of Storey and Tibshirani 2003 \cite{Storey:2003cj}.

\begin{equation}
p_{m} = 1 - \frac{\sum_{r}^{R}|t_{m}| > |t_{mr}^{bs}|}{R}\label{ch-quant_anal:bspval}
\end{equation}

\subsubsection{A metabolomic signature of protein eating in pancreatic tumors}

Overall, the concentrations of 57 metabolites were significantly enriched or depleted in pancreatic cancer tissue relative to adjacent tissue (q-value $<$ 0.05; \hyperref[ch-quant_anal:tnboxplot]{Figure \ref{ch-quant_anal:tnboxplot}}).  In agreement with an increase in tumor glycolytic flux (i.e. the Warburg effect), glucose is specifically depleted in tumors compared to benign tissue, while the fermentative byproduct lactate is enriched.  The behavior of amino acids in tumor samples was far more unexpected.  We initially predicted that the rate of protein synthesis in tumors would at least partially be limited by the rate at which essential amino acids could be supplied by the circulation, while non-essential amino acids that can be synthesized \textit{de novo} in the tumor would not limit growth. The metabolomics data contradicts this assumption; non-essential amino acids appear to partially limit tumor growth, while essential amino acids accumulate. 

\begin{figure}[h!]
\begin{center}
\includegraphics[width=1\textwidth]{ch-quant_anal/Figures/TN_boxplot.pdf}
\caption[Metabolite accumulation or depletion in pancreatic cancer]{Metabolite accumulation or depletion in pancreatic cancer. 57 metabolites with significantly different levels between tumor and paired benign adjacent tissues (FDR $<$ 0.05). Y-axis is the ratio of metabolite concentrations in the tumor to benign adjacent tissue. Vertical lines represent range, boxes show interquartile range, and horizontal lines are the median across the 49 patients.}
\label{ch-quant_anal:tnboxplot}
\end{center}
\end{figure}

Identifying this discrepancy helped our group discover a previously unappreciated route by which tumors can acquire nutrients to supplement their growth requirements.  In both primary tumors and cell culture, tumors activate macropinocytosis; this allows for the bulk uptake of extracellular protein that can be broken down to provide extra amino acids beyond what could be acquired from the circulation. Because protein catabolism yields amino acids in the ratio found in protein, essential amino acids, which can only be used for protein synthesis, accumulate. Non-essential amino acids that can be used for other purposes are depleted.


\subsection{Genetic variation affecting metabolite concentrations in yeast \label{ch:quant_analysis:mQTL}}

In microbes, fitness is tied to growth rate, which is largely established as a byproduct of biosynthetic fluxes \cite{Dykhuizen:1987uq, Edwards:2001hj}. An important outstanding question in metabolism research is the mechanism by which, and the extent to which, segregating genetic variation impacts metabolism and fitness.  From analysis of resting genetic variation, it is clear that central carbon metabolism is under strong purifying selection \cite{Greenberg:2008uy}. This suggests that natural genetic variation impacting enzyme V$_{max}$ is constrained within a relatively narrow range of activities. The functional consequences of excessive or insufficient enzyme activity are generally changes in metabolite pool levels that naturally buffer genetic perturbations \cite{Fendt:2010gr} and provide intrinsic canalisation to the metabolic network. Fitness is compromised when enzyme activity is pushed outside the bounds that can be absorbed by metabolite pools or when an enzyme is inappropriately regulated through either changes in expression or allostery. 

To investigate how genetic variation impacts yeast metabolism, the relative concentrations of 74 metabolites were measured using LC-MS for over 100 yeast segregants derived from a cross between a laboratory strain (BY4716; BY) and a vineyard strain (RM11-1a; RM). These parental strains vary at $\sim$0.6\% of base pairs. Consistent with this extensive variation, the transcriptome, proteome and chemical resistances of BY$\times$RM segregants vary greatly \cite{Brem:2005gh, Foss:2007ej, Bloom:2013bq}. Using non-parametric linkage analysis \cite{Broman:2003wq}, a total of 34 metabolites were associated with at least one quantitative trait locus (QTL); in total 52 QTLs were detected.  These metabolite QTLs (mQTLs) were not randomly distributed through the genome, rather they were preferentially located near genes that encode enzymes in the metabolite's pathway. In addition, mQTLs were clustered into eight hotspots where three or more metabolites were linked to a common locus. Six of these mQTL hotspots overlapped previously detected expression QTL and protein QTL hotspots (\hyperref[ch-quant_anal:qtlHotspots]{Figure \ref{ch-quant_anal:qtlHotspots}}). Of these six hotspots, the genetic basis of three hotspots was investigated through allele-swaps, in which the BY allele was testing in an RM background and the RM allele was tested in a BY background. Using this approach, one engineered and three natural polymorphisms were largely sufficient to reproduce differences between the two parental strains in hotspot metabolites. 

\begin{figure}[h!]
\begin{center}
\includegraphics[width=1\textwidth]{ch-quant_anal/Figures/allQTLhotspots.pdf}
\caption[Transcript, protein and metabolite QTL linkages]{Transcript, protein and metabolite QTL linkages. The number of metabolites with levels linked to a given region of the genome is shown along with linkages to transcript and protein levels \cite{Brem:2005gh, Foss:2007ej}. Chromosomes are distinguished by color and are arranged numerically. To determine the extent to which species are preferentially linked to one or a few hotspots, the cutoff for such preferential linkage is shown as a horizontal black line.}
\label{ch-quant_anal:qtlHotspots}
\end{center}
\end{figure}

Linking metabolite abundance to genetic heterogeneity across segregants assumes that there is substantial genetic variation affecting metabolite levels in the first place.  Previous estimates of broad-sense heritability \cite{Lynch:1998vx} in \textit{A. thaliana} have suggested moderate heritability of metabolite traits across globally-distributed strains \cite{Keurentjes:2006ik}, while segregants exhibited substantially lower heritability of metabolite traits than expression traits (an average of 25\% and 65\%, respectively) \cite{Rowe:2008ty, West:2006bk}.  To calculate broad-sense heritability for each metabolite, segregants with two quantifiable biological replicates were isolated and the variance within replicates was compared to the total across all samples.  This effectively subtracts the environmental variance from the total phenotypic variance to yield the genetic variance.  The ratio of genetic variance to phenotypic variance is the broad sense heritability (\hyperref[heriteq]{Equation \ref{heriteq}}).

\large{
\begin{align}
\hat{\sigma^{2}_{s}} &= \frac{\sum_{r = 1}^{2}\left(X_{sr} - \overline{X}_{s}\right)^2}{2} \cdot \frac{2}{2 - 1}\notag\\
H^{2} &= 1 -  \frac{\sum_{s}^{S}2\hat{\sigma^{2}_{s}}}{\sum_{s}^{S}\sum_{r = 1}^{2}(X_{sr} - \overline{X})^2}\label{heriteq}
\end{align}
}

We found extensive heritable variation of metabolite abundance in this study, with an average broad-sense heritability of 62\%.  This indicates that there are likely larger segregating metabolic differences between BY \& RM than within the Bay$\times$Sha \textit{A. thaliana} cross. Greater levels of heritability across metabolites are associated with an increased number of detected mQTLs (p = 0.014); this is evident in \hyperref[ch-quant_anal:metHeritability]{Figure \ref{ch-quant_anal:metHeritability}}, which shows linkage numbers as a function of heritability. The association between the number of QTLs found for a metabolite and the metabolite's heritability was determined by modeling the number of detected QTLs as an approximately poisson trait that could be predicted using poisson regression \cite{Cameron:2013tp}. The impact of these QTLs can be noted by determining the fraction of the variance in metabolite abundance that is explained using QTL genotypes (\hyperref[ch-quant_anal:qtlEffects]{Figure \ref{ch-quant_anal:qtlEffects}}).

\begin{figure}[h!]
\begin{center}
\includegraphics[width=0.8\textwidth]{ch-quant_anal/Figures/metHeritability.pdf}
\caption[Distribution of broad sense heritability (H$^{2}$) across measured metabolites]{Distribution of broad sense heritability (H$^{2}$) across measured metabolites. Each circle represents a single metabolite, colored according to how many QTLs are associated with its abundance.  114 metabolites are shown: 74 known metabolites with 52 detected mQTL and 42 unknown metabolites (with known m/z, but unknown identity), associated with 20 additional mQTLs.}
\label{ch-quant_anal:metHeritability}
\end{center}
\end{figure}

\begin{figure}[h!]
\begin{center}
\includegraphics[width=1\textwidth]{ch-quant_anal/Figures/qtlEffects.pdf}
\caption[Fraction of broad-sense heritability explained by identified mQTLs]{Fraction of broad-sense heritability explained by identified mQTLs.  Each stacked bar represents a single metabolite that was significantly associated with at least one locus.  The height of the bar is the broad-sense heritability of the metabolite's abundance, and the coloration partitions this heritability into unexplained heritability (gray) and the effects of each mapped QTL (colors).  Three examples are given to demonstrate the variable effect sizes observed across metabolites.  The distribution of metabolite abundances for a genotype is shown as a violin plot, and a 95\% confidence interval for the median of each genotype is reported with error bars.  This confidence interval was determined using a percentile bootstrapping method \cite{Davison:1997vn}.}
\label{ch-quant_anal:qtlEffects}
\end{center}
\end{figure}

Effect sizes and the total fraction of heritability explained vary greatly across metabolites; some mQTLs explain the majority of heritable variation. Others collectively account for a sizable portion through the joint additive effects of multiple loci and others still explain little of the total variance.  It is clear that most metabolite concentrations are modestly heritable, although this calculation is likely inflated due to experimental conflation of segregant and day effects.  Despite this heritability, for most metabolites, only a small fraction of variation can be explained by the detected QTLs.  The low fraction of explained variation could be due to insufficient power to detect additive effects, as has been noted when investigating the genetic basis of variable chemical resistance in BY$\times$RM \cite{Bloom:2013bq}.  An alternative explanation is that metabolites possess a significantly lower narrow-sense heritability (h$^{2}$) than broad-sense heritability (H$^{2}$) because of the nonlinear relationships between pathway fluxes, metabolite concentrations and enzyme activity \cite{Kacser:1973fe, Rowe:2008ty}. Unexplained variation in metabolite concentrations is also likely strongly influenced by differences in culture conditions and technical challenges in reproducibly measuring LC-MS data. 

Most genetic variation that impacts metabolite concentrations only affects a single metabolite.  Given that metabolites are preferentially linked to pathway enzymes, these linkages may reflect the variable activity or expression of a single enzyme. Such genetic perturbations would results in a compensatory change in metabolite concentration that helps to maintain constant flux \cite{Fendt:2010gr}. It is likely that even such modest variants incur some fitness cost, because the large effective population size of yeast ($N_{e} \sim 1.36 \times 10^{7}$) results in the purging of even modestly deleterious variation through selection \cite{Hartl:2007fy,Ohta:1973ub,Wagner:2005cn}. Considering polymorphisms with larger effects, which are unlikely to be readily buffered, we detected QTL hotspots that either specifically alter metabolism or jointly affect transcript, protein and metabolite levels.  Through either large changes in a specific enzyme or altered activity in many enzymes, the genetic variants underlying these hotspots likely significantly impact fitness. This is because such alterations in pathway metabolites likely parallel overt changes in pathway flux \cite{Dykhuizen:1987uq}. Such meaningful differences in metabolic behavior may be tailored to the distinct ecological niches BY and RM occupy, allowing stable maintenance of genetic variation by balancing selection \cite{Hartl:2007fy}. Code supporting my specific contributions to this project is publicly available on \href{https://github.com/shackett/mQTL}{GitHub}.

\subsection{Quantitative proteomics elucidates targets of autophagy}
  
 Autophagy is one of the mechanisms that allows cells to degrade damaged proteins and organelles.  In cancer cells, upregulation of this pathway allows its cooption as a metabolic source of amino acids, thus allowing the proteome to be redistributed \cite{Rabinowitz:2010fx}.  Despite autophagy's importance in cancer, it is not clear which proteins autophagy primarily affects.  To investigate this question, the relative proteome composition of wild-type (WT; Atg5$^{+/+}$) and  autophagy-deficient (Atg5$^{-/-}$) HRas$^{G12V}$-transformed immortalized baby mouse kidney epithelial (iBMK) cells \cite{Guo:2011ba} were compared during normal growth and following either three or five hours of starvation.  We primarily aimed for a fair comparison between autophagy-competent and defective cells at each growth condition; accordingly, at each time point, autophagy-deficient and wild-type samples were labeled using SILAC and were combined \cite{Ong:2002tf}. This way, they could be simultaneously measured and discriminated by virtue of changes in amino acid molecular weight.  Each combined SILAC sample was digested into peptides using trypsin and then fractionated using either strong cation exchange (SCX) or off-gel fractionation (OG).  These two samples, which should be considered technical replicates, were analyzed by LC-MS/MS.  
 
To identify and quantify the peptides present in each of the six samples (time 0, 3, $\&$ 5 hours and SCX versus OG fractionation), samples were separately analyzed using two analytical platforms: MaxQuant \cite{Cox:2008ir} and Proteome Discoverer (Thermo-Fisher; Waltham, MA). From the abundances of peptides, a log$_{2}$ ratio indicating relative abundances of each proteins was calculated as the median of all SILAC log-ratios across unambiguously matching peptides with heavy (Atg5$^{-/-}$) and light (Atg5$^{+/+}$) quantifiable SILAC signal intensities in a given sample. In order to normalize these log protein ratios to account for differences in sample loading, the median of these values was set to zero \cite{Cox:2008ir}, effectively normalizing each sample based on its average signal. The proteins abundances considered were drawn from the union of proteins identified using Proteome Discoverer and MaxQuant, giving precedence to Proteome Discoverer in the intersection. In total 7,184 proteins were quantified, which represented around 25$\%$ of the total mouse proteome.
 
To identify both linear changes in the log of the ratio of autophagy-deficient to WT cells over the course of starvation and non-linear changes in abundance, two regression models were formulated. Therein, time was either used as a linear (\hyperref[ch-quant_anal:autophagyEqtn1]{Equation \ref{ch-quant_anal:autophagyEqtn1}}) or a categorical predictor (\hyperref[ch-quant_anal:autophagyEqtn2]{Equation \ref{ch-quant_anal:autophagyEqtn2}}) of the abundance of a protein \textit{i} in a sample \textit{j}.

\begin{subequations}
\begin{align}
Y_{ij} = \alpha_{i} + t\beta_{ij} + \epsilon_{ij} \label{ch-quant_anal:autophagyEqtn1}\\
Y_{ij} = \beta^{0h}_{ij} + \beta^{3h}_{ij} + \beta^{5h}_{ij} + \epsilon_{ij} \label{ch-quant_anal:autophagyEqtn2}
\end{align}
\end{subequations}

For the linear regression model (\hyperref[ch-quant_anal:autophagyEqtn1]{Equation \ref{ch-quant_anal:autophagyEqtn1}}), t-statistics were generated for each protein to reflect the departure of the initial log$_2$-ratio of the protein from zero as well as the change in this relative abundance over time. Using the categorical regression model (\hyperref[ch-quant_anal:autophagyEqtn2]{Equation \ref{ch-quant_anal:autophagyEqtn2}}), the 3  and 5 hour time points were jointly compared to the initial relative abundances using ANOVA. For the analyses of both models, only proteins that were quantifiable in at least five out of the six conditions (namely, 0, 3, $\&$ 5 hour samples fractionated by SCX; and 0, 3, $\&$ 5 hour samples fractionated by OG) were included in the analysis. 

From this statistical analysis, it is clear that while our experimental design was geared towards identifying how autophagy specifically remodels the proteome during starvation, large differences between autophagy-competent and autophagy-deficient cells exist before starvation begins. These differences persist at a similar magnitude throughout starvation.  Because the standing level of many proteins is influenced by autophagy, a concise way to summarize this effect is by using a t-statistic. In doing so, the peptide-wise fitted time zero intercept of \hyperref[ch-quant_anal:autophagyEqtn1]{Equation \ref{ch-quant_anal:autophagyEqtn1}} is compared to the expected intercept of zero (equivalent to equal abundance of the two samples), accounting for the standard error of the intercept's estimate (\hyperref[ch-quant_anal:autophagyProteomics]{Figure \ref{ch-quant_anal:autophagyProteomics}}).

\begin{figure}[h!]
\begin{center}
\includegraphics[width=0.6\textwidth]{ch-quant_anal/Figures/autophagyProteomics.pdf}
\caption[Proteomic alterations due to loss of autophagy]{Proteomic alterations due to loss of autophagy. Volcano plot showing statistical significance of pre-starvation differences in protein abundances due to autophagy function. Proteins whose abundance is significantly altered due to autophagy are shown. A total of 1,638 proteins were preferentially degraded in autophagy-competent cells (yellow), while 1,543 were selectively preserved (blue).}
\label{ch-quant_anal:autophagyProteomics}
\end{center}
\end{figure}

In total, 3,181 proteins are significantly enriched or depleted in autophagy-deficient cells under both normal and starvation conditions. 1,638 of these proteins are elevated in autophagy-deficient cells, and thus, serve as autophagy substrates. 1,543 proteins are selectively preserved by autophagy.  Gene set enrichment analysis (GSEA) of proteins elevated and depleted in autophagy reveals that vesicle trafficking proteins are maintained by autophagy, whereas proteins involved in immunity are among those most actively degraded.


