\section{Conclusion}

It seems like science fiction to suggest that, the gross complexity of an organism as complicated as \textit{homo sapiens} could be reduced to a model of physiology that is sufficient to fully understand how genetic variation and environmental exposures shape phenotypic variation at every scale. However, I firmly believe that the field is poised for just such a model to be created for a suitable microbe. Such a model would required detailed experimental measurements of nearly all cellular species, a technical requirement which is becoming increasingly more feasible through the ever increasing accuracy and breadth of sequencing, mass spectrometry and high-throughput microscopy. These measurements alone can characterize the static state of an organism, but they can not tell us about the relationships between chemical species which lead to the emergent property of physiology. 

To gain insights into how species interact, we must note how they covary. Generating such variation can be done through a combination of genetic and environmental perturbations which would ideally generate as much orthogonal physiological variation as possible. Regardless of how many conditions could be practically assessed, the large space of possible interactions between variables would challenge analysis. To guide the create of models, core principles of molecular biology and biochemistry can serve as a scaffold to lay biological information. I believe that my thesis research espouses this philosophy, provides clear practical examples of how experimental data can be used to guide bottom-up reconstruction of complex systems.

When integrating multiple types of data, it is all the more important that each dataset be analyzed appropriately. Without such consideration, the added noise from combining multiple datasets will further hide meaningful biological signal and make it difficult to know whether a given proposed model is adequate. Towards this goal, during Chapter \ref{ch-quant_anal}, I discussed practical considerations when using mass spectrometry data.  While mass spectrometry is an ideal platform for quantifying important physiological variables, such as metabolites, peptides/proteins and lipids, in some cases experimental data does not agree with the assumption of log-normal variability. When analyzing variation in a single class of species, such as metabolites, deviations can be dealt with through non-parametric approaches, but when a dataset is meant to be integrate it into systems-level models, summarizing data based on parametric assumptions, is of great utility.  This is because, when creating models, it is important frame relate uncertainty in model outputs in light of uncertainty in model inputs. The multivariate delta method used in Chapter \ref{ch:simmer} be particularly useful in this regard. In this chapter, I tested the assumptions of mass spectrometry data analysis using three moderately sized datasets, and found that these datasets conformed to log-normal assumptions to a variable degree. In cases where a standard log-normal model is not valid, I further discussed how heteroscedastic models of parametric variation may allow for broader application of log-normal models to mass spectrometry data.

Creating a multi-omic dataset which combines measurements of transcript, protein and metabolite abundances with inferred metabolic fluxes across 25 physiologically and metabolically heterogeneous states enabled me to answer study interactions which are inherently multi-omic.  In each case, appropriate experiments allowed me to reduce a genome-scale question into a set of sub-problems that could be addressed at a more focused and tractable scale; either at the level of a metabolic reaction or a single gene.  

\subsection{Understanding the structure and regulation of metabolism}

During Chapter \ref{ch-simmer} I used an integrative `omic approach to model metabolism by virtue of reaction-level interactions between metabolites, enzymes and consequent flux. For each of \textcolor{red}{n} reactions, an approximate model of reaction kinetics was inferred based on balancing the plausibility of each model, $Pr(Model)$, with each kinetic model's quantitative support, $Pr(Data | Model)$. This approach termed SIMMER (\underline{S}ystematic \underline{I}nference of \underline{M}eaningful \underline{M}etabolic \underline{E}nzyme \underline{R}egulation), identified \textcolor{red}{n} instances of allostery which were strongly supported in yeast.  Many of these regulatory mechanisms were strongly supported based on past literature; others could be experimentally validated through \textit{in vitro} biochemistry; and still other were likely false positives due to experimental insufficiencies.

SIMMER indicates whether Michaelis-Menten kinetics is an adequate model of reaction kinetics and if not, what regulation (if any) is best supported. Model inference naturally entails assessing whether the physiological variability in all reaction species can collectively explain changes in flux. To partition this total control into the marginal influence of each species, I motivated the concept of metabolic leverage, whereby the total metabolic leverage of a reaction is decomposed into the influence of individual species by virtue of their total physiological variability and the impact of this variation (average elasticity). Summarizing reactions this way, two classes of reactions clearly emerge.  Variable flux through kinetically reversible reactions is primarily accomplished through changes in substrates and/or product concentrations, with only minor contributions due to changes in enzyme levels.  In contrast, variable flux through kinetically irreversible reactions is attributable to joint variation in substrates, enzymes and possibly regulators.

The common trends in metabolic leverage shed light on how yeast are able to control their metabolism to grow optimally in radically differing environments. From analysis of metabolic leverage, reversible and irreversible reactions serve distinct kinetic functions. Variable flux through reversible reactions is primarily driven by changes in substrate and product concentrations allowing these reactions to serve as a bridge between irreversible reactions. Irreversible reactions are usually affected by one or more external control mechanisms, either changes in expression wired to the transcriptional/post-transcriptional regulation or metabolic regulation which allows for coordination of pathway fluxes to either metabolic demands.

From the analysis of metabolic leverage it appears that the metabolome inherently yields an appropriate distribution of flux while the changes in protein abundance, while important, primarily serve to improve efficiency (cornish bowden). This conclusion helps to explain notorious challenges in altering pathway fluxes through altering the expression of a single or even several of pathway enzymes. The importance of this metabolic control paradigm becomes clear when one considers how metabolic robustness is maintained in the face of genetic and or non-metabolic environmental perturbations.

These control principles also shed light on how the metabolome can intrinsically canalize metabolism to genetic perturbations \cite{Larhlimi:2011ds}. If the activity of enzymes catalyzing reversible reactions is perturbed, the balance of substrates and products will shift to appropriately counteract the induced effect (link). When the activity of irreversible reactions is altered, the metabolic consequences of their regulation can be generally framed in light of how they are regulated as summarized by the substrate/product, enzyme, regulator metabolic leverage ternary diagram. For enzymes residing near the center of this diagram, altered enzyme activity would be partially absorbed through changes in substrate abundance. If this change is insufficient, then in the case of feedback control, a mismatch between pathway production and pathway demands will alter the level of pathway metabolites until feedback control attenuates the changes in pathway production.

While the metabolic network has generally evolved to tune the fluxes of major control points to the demand of these reactions by virtue of changes in metabolite concentrations, there is a cost to such metabolomic fine-tuning.  As regulation of flux due to metabolites is either felt through metabolite reversibility or via the transfer of metabolic control from the metabolic source onto the metabolic sink, such regulation inherently requires the variable accumulation of metabolites. In microbes, the maintenance of metabolite pools which purely assist kinetics does incur a fitness cost, since these pools could be better incorporated into biomass to improve growth. In some cases this cost could be massive, such as if feedback-control on a first-committed step was ablated, accumulation of products would only affect the pathway production once the reaction was in approximate thermodynamic equilibrium. The fitness cost of metabolomic fine-tuning helps to explain why many metabolic enzymes are under strong purifying selection \cite{Greenberg:2008uy}, despite only a minority of enzymes being strongly regulated through abundance or exerting meaningful flux control.

In yeast, under natural conditions, metabolism appears to be well regulated. Cases of substantial dysregulation could be manifest as large changes in product concentrations (as is seen in the uracil auxotrophy), or if allostery was the primary determinant of reaction flux. Because most reactions are primarily regulated through substrate(s) and enzyme concentrations, with additional fine-tuning of flux through physiologically variable allostery, the accumulation of regulators is likely not a huge fitness cost.

\subsection{Understanding how enzyme levels are regulated}

During Chapter \ref{ch-pt_compare}, I discussed another important outstanding question in systems biology, how variable levels of proteins are produced across diverse growth conditions. Studying each gene independently, I determined whether nutrient induced changes in protein abundance can be explained based on changes in their cognate transcript, when departures between proteins and transcripts may be due to post-transcriptional regulation, and when these departures are likely due to experimental noise. This analysis showed that while strong transcriptional changes propagate into corresponding changes in protein abundance, additional layers of post-transcriptional control are relevant.
