\section{Conclusion}

It may seem like science fiction to suggest that human complexity could be reduced to a physiological model where a multitude of genetic variants and environmental exposures could jointly inform numerous complex phenotypes. However, I firmly believe that the field is poised to create just such a model for a suitable microbe. This endeavor will require detailed experimental measurements of nearly all cellular species, a technical feat which is becoming increasingly feasible through advancements in sequencing, mass spectrometry and other high-throughput methods. These measurements alone can characterize the static state of an organism, but they can not tell us about the relationships between chemical species which lead to the emergent property of physiology. 

To learn how species interact, one important strategy in genomics has been detecting their physical interaction using library-based methods \cite{LiebermanAiden:2009jz, Fields:1989dm, Orsak:2012ci, Johnson:2007fh}.  While this approach can suggest physical associations, with few exceptions \cite{Reynolds:2011gs} it is unclear whether such associations are functionally significant \cite{Nandy:2010ej, Scheer:2011df}. 

To establish the relevance of interactions, physiological covariation can be used to suggest dependence, but this is a challenging model inference problem due to model degeneracy. To associate variables, extensive orthogonal variation must be generated through a combination of genetic and/or environmental perturbations \cite{Greenberg:2011jf}. However, regardless of how many conditions can be practically assessed, the large space of possible interactions between variables will challenge analysis. To guide the create of models, core principles of molecular biology and biochemistry can serve as a scaffold to lay biological information. I believe that my thesis research espouses this philosophy, providing clear practical examples of how experimental data can be used to guide bottom-up reconstruction of complex systems.

When integrating multiple types of data, it is all the more important to generate each dataset in a way that maximizes signal and to analyze the resulting data using appropriate statistical methods. Without such consideration, the added noise from combining multiple datasets will mask meaningful biological signal or inappropriate methods could make it difficult to determine whether a model is consistent with the input data. Towards this goal, during \hyperref[ch:quant_anal]{Chapter \ref{ch:quant_anal}}, I discussed practical considerations when using mass spectrometry data.  While mass spectrometry is an ideal platform for quantifying important physiological variables, such as metabolites, peptides/proteins and lipids, in some cases experimental data does not conform to the assumption of log-normality. In cases where a standard log-normal model is not valid, I further discussed how heteroscedastic models of parametric variation may allow for broader application of log-normal models to mass spectrometry data. When analyzing variation in a single class of species, such as metabolites, deviations can be dealt with through non-parametric approaches, but when a dataset is meant to be integrate it into systems-level models, summarizing data based on parametric assumptions, is of great utility.  This is because, when creating models, it is important that uncertainty in model outputs corresponds to uncertainty in model inputs. The multivariate delta method used in \textcolor{red}{sectionRef} \hyperref[ch:simmer]{Chapter \ref{ch:simmer}} is particularly useful in this regard \cite{Lynch:1998vx}.

By generating a multi-omic dataset where four major classes of biomolecules and processes (transcripts, proteins, metabolites and fluxes) were quantified across 25 physiologically and metabolically heterogeneous states, I was able to investigate two classes of multi-omic interactions. Enzymes, metabolites and flux collide at the level of metabolic reactions, while the process of translation leads to a fundamental link between transcripts and proteins. In each case, appropriate experiments allowed me to reduce a genome-scale question into a set of sub-problems that could be addressed at a more focused and tractable scale; either at the level of a metabolic reaction or a single gene.

\subsection{Understanding the structure and regulation of metabolism}

During \hyperref[ch:simmer]{Chapter \ref{ch:simmer}} I used an integrative `omic approach to model metabolism by virtue of reaction-level interactions between metabolites, enzymes and consequent flux. For each of \textcolor{red}{n} reactions, an approximate model of reaction kinetics was inferred based on balancing the plausibility of each model, $Pr(Model)$, with each kinetic model's quantitative support, $Pr(Data | Model)$. This approach termed SIMMER (\underline{S}ystematic \underline{I}nference of \underline{M}eaningful \underline{M}etabolic \underline{E}nzyme \underline{R}egulation), identified \textcolor{red}{n} instances of allostery which were strongly supported in yeast.  Many of these regulatory mechanisms were strongly supported based on past literature; others could be experimentally validated through \textit{in vitro} biochemistry; and still other were likely false positives due to experimental insufficiencies.

\subsubsection{Characterizing physiological regulation in yeast metabolism}

Using these simple models we determined that for 44 of 71 reactions simple Michaelis-Menten kinetics including at most one regulators could accurately relate metabolite and enzyme concentrations to reaction flux. This is impressive because mechanistic modeling of the kinetics of well-studied reactions rarely reduces to Michaelis-Menten kinetics, and often many regulators are thought to collectively regulate activity \cite{Hill:1977vm}.  Under assay conditions, significant deviations from Michaelis-Menten kinetics may exist and the activity of regulators can be demonstrated; for the purpose of understanding flux control, a reaction form does not need to be principled mechanistically, but rather must only account for an approximately correct relationship between species and resulting flux across the physiological conditions investigated \cite{Fell:1997wg}. Across the diverse conditions that we studied, Michaelis-Menten kinetics is generally appropriate and a single regulator usually dominates even for regulatory hubs such as phosphofructokinase, pyurvate kinase and DAHP synthase.  

For each reaction, the possible regulators that we tested were aggregated from reports across all domains of life, yet the most likely regulator for well-studied reactions generally agreed with what we already know about yeast.  Negative feedbacks of anabolic pathways are the best studied cases of metabolic regulation because the logic of these pathways is clear; an end-product inhibits the first committed step for its synthesis.  By allowing the supply of an end-product to sense how quickly it is being utilized, control of pathway flux can be transferred from synthesis to utilization \cite{CornishBowden:1995fy}. Because these regulatory events are so predictable, and the end-product inhibitor can generally be easily isolated, most of these regulatory interactions were demonstrated \textit{in vitro} over fifty years ago, and it is encouraging that they are kinetically important across normal growth conditions.  Glycolysis has also been extremely well studied in yeast both because it is the central path by which nearly all energy and biomass is made and because researchers have sought to understand how metabolism is rewired to accomplish the diauxic shift \cite{Zampar:2013fr}. As such, glycolysis is one of the few pathways for which we have any knowledge of flux control.  In yeast, the rate of fermentation is controlled by transcriptional alteration of glucose uptake as well as by phosphofructokinase activity.  The latter is thought to be impacted by several regulators, particularly by activation by fructose 2,6-bisphopshate \cite{Cortassa:1994is, vanEunen:2012cr}. While we did not measure fructose 2,6-bisphosphate in this study, inhbition by citrate was necessary to predict flux through phosphofructokinase. Because we could not test the role of fructose 2,6-bisphopshate, we cannot conclude whether citrate is a meaningful inhibitor or whether it is merely anticorrelated with fructose 2,6-bisphosphate. Pyruvate kinase is also thought to be an important regulated step of glycolysis; allosteric activation of pyruvate kinase by fructose 1,6-bisphosphate controls the concentrations of metabolites in lower glycolysis, allowing lower glycolysis to keep pace with upper glycolysis when cells are faced with fluctuating glucose concentrations \cite{Xu:2012gg}. Outside of these canonical examples, less is known about metabolic regulation because the logic of how most reactions should be regulated is unclear. Consequently, hypothesis-driven research is less fruitful. Because we can test many possible regulatory models and interpret novel predictions in the context of how they alter flux across conditions, our approach is a powerful way to identify novel regulation with little prior knowledge.  This approach was successfully used, for example, to discover that alanine is a physiological inhibitor of ornithine transcarbamylase (OTCase: Arg3).  Because OTCase and aspartate transcarbamylase (ATCase) compete for a common metabolite carbamoyl phosphate, inhibition of OTCase by alanine can favor production of pyrimidines when aliphatic amino acids are abundant.  This provides a logical route by which amino acid concentrations can favor rRNA synthesis and thereby use amino acid reserves for protein synthesis.  

\subsubsection{How coordination of the metabolism and proteome leads to nutrient-dependent growth control}

SIMMER indicates whether Michaelis-Menten kinetics is an adequate model of reaction kinetics and if not, what regulation (if any) is best supported. Model inference naturally entails assessing whether the physiological variability in all reaction species can collectively explain changes in flux. To partition the joint influence of all species into the marginal influence of each specie, I motivated the concept of \textit{metabolic leverage}, where the relative influence of each specie is governed by how much the specie varies and how this variation impacts flux. Summarizing reactions this way, two classes of reactions clearly emerge.  Variable flux through kinetically reversible reactions is primarily accomplished through changes in substrates and/or product concentrations, with only minor contributions due to changes in enzyme levels.  In contrast, variable flux through kinetically irreversible reactions is attributable to joint variation in substrates, enzymes and possibly regulators.

Common trends in metabolic leverage shed light on how yeast are able to control their metabolism to grow optimally in radically differing environments. From analysis of metabolic leverage, reversible and irreversible reactions serve distinct kinetic functions. Variable flux through reversible reactions is primarily driven by changes in substrate and product concentrations allowing these reactions to serve as a bridge between irreversible reactions. Irreversible reactions are usually affected by one or more external control mechanisms, either changes in expression wired to the transcriptional/post-transcriptional regulation or metabolic regulation which allows for coordination of pathway fluxes to either metabolic demands. From the analysis of metabolic leverage it appears that the metabolome inherently yields an appropriate distribution of flux while the changes in protein abundance, while important, primarily serve to improve efficiency. This conclusion helps to explain notorious challenges in altering pathway fluxes through altering the expression of a single or even multiple pathway enzymes \cite{CornishBowden:1995fy,Schaaff:1989dl,Hauf:2000vu,Fell:1997wg}. 

The importance of this metabolic control paradigm becomes clear when one considers how metabolic robustness is maintained in the face of genetic and or non-metabolic environmental perturbations \cite{Larhlimi:2011ds}. If the activity of enzymes catalyzing reversible reactions is perturbed, the balance of substrates and products will shift to appropriately counteract the induced effect \cite{Fendt:2010gr}. When the activity of irreversible reactions is altered, the metabolic consequences of their regulation can be generally framed in light of how they are regulated as summarized by the substrate/product, enzyme, regulator metabolic leverage ternary diagram. For enzymes residing near the center of this diagram, altered enzyme activity would be partially absorbed through changes in substrate abundance \cite{Fendt:2010gr}. If this change is insufficient, then in the case of feedback control, a mismatch between pathway production and pathway demands will alter the level of pathway metabolites until feedback control attenuates the changes in pathway production.

While the metabolic network has generally evolved to tune the fluxes of major control points to the demand of these reactions by virtue of changes in metabolite concentrations, there is a cost to such metabolomic fine-tuning.  As regulation of flux due to metabolites is either felt through metabolite reversibility or via the transfer of metabolic control from the metabolic source onto the metabolic sink \cite{Fell:1997wg}, such regulation inherently requires the variable accumulation of metabolites. In microbes, the maintenance of metabolite pools which purely assist kinetics does incur a fitness cost, since these pools could be better incorporated into biomass to improve growth. In some cases this cost could be massive, such as if feedback-control on a first-committed step was ablated, accumulation of products would only affect the pathway production once the reaction was in approximate thermodynamic equilibrium \cite{CornishBowden:1995fy}. The fitness cost of metabolomic fine-tuning helps to explain why many metabolic enzymes are under strong purifying selection \cite{Greenberg:2008uy}, despite only a minority of enzymes being strongly regulated through abundance or exerting meaningful flux control.

In yeast, under natural conditions, metabolism appears to be well regulated. Cases of substantial dysregulation could be manifest as large changes in product concentrations (as is seen in the uracil auxotrophy), or if allostery was the primary determinant of reaction flux. Because most reactions are primarily regulated through substrate(s) and enzyme concentrations, allostery primarily fine-tunes approximately correct inputs, resulting in only minimal excess accumulation of pathway products

\subsection{Understanding how enzyme levels are regulated}

While it is routine in the field to treat transcript abundance as a surrogate for protein abundance and gene function, systematic differences between transcripts and proteins, illustrated during \hyperref[ch:pt_compare]{Chapter \ref{ch:pt_compare}}, suggest that it is unwise to assume such a false equivalence. Inspecting variation in proteins-per-transcripts suggests that post-transcriptional regulation through control of translational efficiency and/or protein degradations supplements the primary transcriptional response to nutrient availability. 

The basal level of protein expression control is through control of transcript abundance.  With the wide-spread adoption of microarrays and later RNAseq, for many organisms, previous experiments have determined under what conditions individual transcripts are elevated or depleted \cite{Edgar:2002tt}. While we still have a far from complete knowledge of how transcript levels are determined, transcriptional control is increasingly being understood based on the activity of transcription factors whose regulatory specificity is governed by cis-regulatory sequences \cite{Boorsma:2008cv, McIsaac:2012da}. Small RNAs serve to further modulate expression by altering transcription and stability across conditions \cite{ValenciaSanchez:2006eb, Guo:2010tv}.

Post-transcriptional regulation is increasingly appreciated as an important layer of control on top of transcript abundance, however most studies to date have focused on between-variation in regulation rather than assessing the extent to which regulation of single gene occurs across conditions. Both average translational efficiency and protein stability have been shown to vary greatly across genes \cite{Belle:2006hv, Ingolia:2011hu}. These processes can be altered across conditions, with translation rate being impacted by both amino acid levels \cite{Klumpp:2009ic} and being modulated through the use of internal ribosome entry sites (IRESs) \cite{Hellen:2001cw} while degradation through specialized pathway also depends upon the environment \cite{Callis:1995ki}. Less work exists which bridges between-gene and between-condition variation by assessing how different genes are regulated across conditions \cite{Brar:2012ig, Jovanovic:2015hp}. 

Comparing transcriptional changes to changes in protein expression across 25 diverse physiological conditions, allows transcriptional and post-transcriptional regulation to be studied under conditions which span much of natural environmental variation.  Studying each gene independently, I determined whether nutrient induced changes in protein abundance can be explained based on changes in their cognate transcript, when departures between proteins and transcripts may be due to post-transcriptional regulation, and when these departures are likely due to experimental noise. This analysis showed that while strong transcriptional changes propagate into corresponding changes in protein abundance, additional layers of post-transcriptional control are relevant.

An important role that post-transcriptional regulation appears to serve in modulating the primary transcriptional response is adjusting the proteome to the availability of specific nutrients. While classical examples of transcription regulation indicate how metabolites can control transcription \cite{Jacob:1961du, Jones:1982dn}, the expression of most transcripts is more strongly affected by growth rate than the availability of specific nutrients.  In comparison, the proteome, appears more similar to the metabolome, with protein levels being altered in response to the specific nutritional needs of the cell. Metabolic effects on post-transcriptional regulation could account for such a divide whether considering variable amino acid levels impacting translation rate \cite{Klumpp:2009ic}, or protein degradation rate responding to nitrogen starvation \textcolor{red}{cite}. By further linking metabolism to protein expression, post-transcriptional regulation appears to serve an important physiological role in growth control. As such, individual proteins appear to be differentially impacted by phenomena which are generally thought to impact all genes similar; whether changes in  


