\section{Future Work}

While the work I have presented can serve as a template for how metabolism and protein expression can be efficiently interrogated, my analysis of these questions is limited by the absence of crucial information and an incomplete survey of physio-genetic variation. As such, when discussing future directions of fruitful research I will primarily focus on how the caveats limiting my current research could be best addressed. From there I will touch upon the real world utility of metabolic and expression models, specifically focusing on how such models could be applied to metabolic engineering to improve the efficiency of microbial compound production.

\subsection{Achieving a quantitative understanding of metabolism}

As discussed during \hyperref[ch:simmer]{Chapter \ref{ch:simmer}}, the kinetics of metabolic reactions can be analyzed \textit{en masse} if we have measured reaction flux and all reaction species across a sufficient array of conditions where alternative models can be statistically discriminated. The success of such an approach requires that one of the tested models is an adequate reflection of true \textit{in vivo} kinetics and that no measurement is systematically biased. In practice, no such guarantees be afforded at this juncture as a substantial list of caveats currently adulterate SIMMER. These limitations largely arise from phenomenon with little demonstrated metabolic significance and thus their specific impacts are largely unknown, however by experimentally addressing each question within the SIMMER framework, a more complete understanding of how metabolism is regulated could be found.

\subsubsection{Metabolomic incompleteness}

One current limitation of the SIMMER methodology is that while metabolomics is able to measure a meaningful fraction of important small molecules, this number is minute when compared to the total number of species in most metabolic reconstructions \textcolor{red}{look up}. Mass spectrometry based metabolomics is best at measuring abundant intracellular metabolites which readily ionize resulting in the preferential ascertainment of metabolites in central carbon metabolism, where consequently SIMMER can be best applied. Because of the incompleteness of the ascertained metabolome, for most reactions, the primary substrate(s) have not been measured and thus SIMMER cannot be intelligently applied. In other cases unmeasured products or minor substrates, such as protons or water, may prevent the identification of an appropriate kinetic model or lead to false positives, such as if a missing product is correlated with an inhibitor.  When such species are missing it is assumed that their concentration is invariant; thus this assumption will only impact the application of SIMMER to the extent that physiological variation in missing species impacts flux.

Measuring a wider array of metabolites would allow the broader application of SIMMER. A partial solution to this limitation will be naturally afforded through the passage of time; as mass spectrometer sensitivity continues to advance at a rapid pace, measurement of scarcer and scarcer species becomes more routine. Improvements in sensitivity are only a partial solution as alternative protocols may be necessary to discriminate chemically similar species, or to measure species that are not reliably detected using standard methods. 

While direct measurement of small molecules is the best solution to account for each small molecule's kinetic impact, for some species measurement will be challenging due to either their low concentration (e.g \textcolor{red}{fubar}), lability (e.g. oxaloacetate), or measurement difficulty (e.g. protons and water). In some cases, such as for water, it might be reasonable to treat such metabolites as invariant, but an interesting alternative solution to this limitation is motivated by use of \textit{hypothetical regulators} during \textcolor{red}{cite section}.  Using this approach, both reaction kinetics and an optimal pattern of regulator abundance (informed by the metabolomics principle components) are simultaneously inferred. An analogous approach could be used to impute the abundance of an unmeasured metabolite, either for a single isolated reaction, or for multiple coupled reaction.  For instance, the relative abundance of oxaloacetate could be inferred based on its shared involvement in mitochondrial malate dehydrogenase, citrate synthase and pyruvate carboxylase. Similarly the concentration of protons in the cell, whose variation across physiological conditions is largely unknown, could be assessed by coupling the numerous cellular protonation reactions. While intriguing, jointly inferring the kinetics of coupled reactions takes away from the scalability of the SIMMER, a problem that could likely be relieved using an Expectation-Maximization-like approach to iterate between solving metabolite abundances through reaction-coupling and optimizing the kinetics of individual reactions conditioning on imputed metabolite abundance. On the more exploratory side, this PC-based imputation approach could be used to explore the kinetics of additional reactions by combining multiple under-characterized reactions with unknown metabolites together into a coupled inference problem. This could be a powerful approach for metabolic exploration as the inference would still be underdetermined and the use of a bayesian approach appropriately conveys uncertainty in individual kinetic parameters and metabolite loadings. Beyond metabolism, elsewhere in genomics and beyond, PC-based imputation may serve as a powerful gap-filling approach.

\subsubsection{Improving the precision of flux estimation}

In order to break metabolic pathways into reaction-level inference problems, reaction flux must be treated as a fixed quantity. Determining flux is challenging, as fluxes are rates rather than physical species that can be directly measured. Techniques to determine these rates have either largely focused on estimating fluxes that maximize growth using Flux Balance Analysis (FBA) \cite{Orth:2010hb} or introducing isotopically labelled metabolites which propagate through metabolism using Metabolic Flux Analysis (MFA) \cite{Zamboni:2009bp}. The needs of inferring fluxes in chemostat culture are somewhat different from standard problems as cells are growing at known rate and metabolic sources and sinks can be readily estimated. Using this information the variant of Flux Balance Analysis (FBA) that I developed should be particularly well suited for estimation of fluxes which are strongly determined by these boundary fluxes. This method focuses on finding a least-squares solution which optimally conforms to experimental data by using quadratic programming, 

Not all fluxes are likely well estimated by virtue of metabolite uptake, excretion and incorporation into biomass components. Based on boundary fluxes alone, it is difficult to estimate flux through pathways with stoichiometrically equivalent routes for flux (such as in 1C metabolism) or through pathways which are primarily effect energy or reducing equivalents rather than uptake, excretion or biomass component synthesis. To better quantify such pathways, future work should supplement experimental boundary fluxes, which can be determined through spectrophotometry and $^{1}$H-NMR with additional experimental measurements which further constrain possible flux.

One set of measurements that could allow for better estimation of glycolytic and TCA flux is respirometry.  In microbes, oxygen consumption is a surrogate for electron transport chain flux, while the respiratory quotient $\left(\sfrac{\nu_{CO_{2}}}{\nu_{O_{2}}}\right)$ is a strong indication of the balance of fermentation versus aerobic respiration \cite{Boer:2003fi,BARFORD:1979ei}. To further refine estimates of intracellular flux techniques from MFA could be applied to chemostat culture. This could be particularly useful for determining the ratio of fluxes between pathways such as between glycolysis and the oxidative pentose phosphate pathway, or between lower-glycolysis and the TCA cycle \textcolor{red}{cite}. While MFA is generally considered a comprehensive framework for the estimation of metabolism-wide fluxes which could incorporate experimental boundary fluxes, estimates of reaction flux could also be incorporated into a quadratic programming based method.

Estimating additional reaction fluxes using respirometry \&/or MFA would allow for better estimation of the expected values of pathway fluxes, but the uncertainty in these measurements could also be used to better characterize the uncertainty in metabolism-wide flux. Since the quadratic penalty is constructed by weighting least-squared deviations by experimental precision analogously to gaussian log-likelihood $\left(\nu^{T}\Sigma^{-1}\nu = \sum_{j = 1}^{J}\frac{\left(x_{j} - \mu_{j}\right)^2}{\sigma^{2}_j}\right)$, uncertainty in flux can by assessing the maximum and minimum flux that could go through each reaction at the solution minimum as well as at sub-optimal solutions which approximate cuts through the likelihood distribution (\hyperref[conclusion-fluxUncertainty]{Figure \ref{conclusion-fluxUncertainty}}). This is conceptually similar to estimating confidence intervals based on the likelihood ratio. SIMMER currently deals with uncertainty in flux using flux variability analysis \textcolor{red}{cite} based on how constrained individual fluxes are at the solution optimum. A solution which more fully deals with uncertainty could be readily incorporated into SIMMER by integrating over the full flux probability density function.  This solution was not used during \hyperref[ch:simmer]{Chapter \ref{ch:simmer}} because the absence of internal fluxes prevented strong constraints on most fluxes at sub-optimal values resulting in pathological flux distributions. 

\begin{figure}[h!]
\begin{center}
\includegraphics[width=0.8\textwidth]{ch-conclusion/fluxVar.pdf}
\caption[Procedure for accounting for flux uncertainty]{Procedure for accounting for flux uncertainty. \textbf{A)} The relationship between the maximum and minimum flux that are possible at a given quadratic penalty / log-likelihood can be used to construct a \textit{de facto} log-likelihood distribution for each reaction's flux. \textbf{B)} The log-likelihood distribution corresponds to a probability distribution, p(v).}
\label{conclusion-fluxUncertainty}
\end{center}
\end{figure}





\subsubsection{Mis-characterization of enzyme activity and kinetic differences between iso-enzymes}

When incorporating enzyme-level information into SIMMER, measurement of protein relative abundance can be accurately applied in the simple case where a single gene produces an enzyme, and each copy of this enzyme is kinetically indistinguishable. Outside of this simple scenario, kinetic inference is complicated by multiple distinct pools of enzymes that catalyze a single reaction. Relevant enzymatic pools whose kinetics could greatly vary include isoenzymes which catalyze the same reaction \textcolor{red}{cite bradley}, enzyme complexes where pools of active enzyme may not be reflected by the abundances of monomers \textcolor{red}{cite}, and cases where post-translational modification of enzymes alters metabolite affinity or activity. 

Because many enzymes do not operate as a monomer, the abundance of each catalytically-active complex that catalyzes a given reaction should be included as a separate distinct ``enzyme'' in SIMMER. While we have an incomplete understanding of how complex abundance and activity relates to the abundance of individual components and, in general monomers form stable complexes, while a stoichiometric misbalance between components results in the excess being degraded \textcolor{red}{cite}. In such simple examples, the abundance of a complex would reflect the abundance of each component monomer.  As the genome scale model \textcolor{red}{model} used contains annotated complex membership, to date complex abundance has been estimated using the precision-weighted mean over all complex components. In some cases such a simplified view of complex formation may be inaccurate, however more sophisticated models of complex formation which estimate complex abundance from proteomics data could easily be incorporated into SIMMER.
 
When multiple complexes or isoenzymes can catalyze a single reaction, we are left with two options; either we can assume that each set of enzymes is kinetically similar or different. If we assume that all pools of enzymes interact with metabolites in a similar manner (shared $k_{d}$ and possible $k_{i}$/$k_{a}$) then the occupancy of each enzyme will be the same ($\sfrac{\nu}{\nu_{max}}$) while $k_{cat}$ will vary to reflect that measured enzyme concentrations are relative and sizes of enzyme pools may differ.  This is the standard approach currently used in SIMMER as this minimizes the total number of fitted kinetic parameters, and in most cases a single complex likely carries the bulk of flux. If we chose to relax this assumption, SIMMER is already  compatible with reaction forms where one or more kinetic parameters (besides $k_{eq}$) differ between isoenzymes or complexes. Treating simple Michaelis-Menten kinetics catalyzed by two isoenzyme (E$_{a}$ and E$_{b}$) as an example, reaction flux would follow \hyperref[Eq:isoenzymeMM]{Equation \ref{Eq:isoenzymeMM}}. DAHP synthase (Aro3/Aro4), the first step of aromatic amino synthesis, is one clear example in yeast where such isoenzyme-specific regulation is likely relevant since Aro3p is specifically inhibited by phenylalanine, while Aro4p is specifically inhibited by tyrosine \cite{Schnappauf:1998ec}. The kinetics of this reaction were specifically tested for isoenzyme-specific regulation, but the support for this regulation was quantitatively weaker than predicted regulation by phenylpyruvate (a phenylalanine precursor) \textcolor{red}{Section}. 

\begin{equation}
\nu = k_{cat}^{E_{a}}\left[E_{a}\right]\frac{\left[S\right]}{\left[S\right] + k_{m}^{E_{a}}} + k_{cat}^{E_{b}}\left[E_{b}\right]\frac{\left[S\right]}{\left[S\right] + k_{m}^{E_{b}}}\label{Eq:isoenzymeMM}
\end{equation}

Pools of enzymes may differ kinetically because they are composed of different proteins, but the kinetics of otherwise identical enzymes or complexes may also differ if they possess different post-translational modifications.  Post-translational modifications (particularly phosphorylation) are rampant in yeast \cite{Fiedler:2009hx}; their effects on metabolism, however, are largely unknown aside from a few phosphorylation events with an implicated kinetic role, particularly in the TCA cycle \cite{Schulz:2014eo}. While post-translational modifications have not been characterized under the measured conditions, SIMMER could be easily adapted to suggest the physiological role of post-translational modifications in a similar manner to testing allostery. To illustrate how such inference could be conducted, we can consider whether assessing the effect of a single phosphorylation event with a putative effect on enzyme activity explains measured flux significantly better than a model without regulation. If we knew the relative abundance of both the modified protein, $\left[E_{p}\right]$, and unmodified protein $\left[E_{u}\right]$, then reaction kinetics may be described by \hyperref[Eq:ptmMM1]{Equation \ref{Eq:ptmMM1}}, which fits one more parameter than the minimal model.

\begin{equation}
\nu = k_{cat}^{p}\left[E_{p}\right]\frac{\left[S\right]}{\left[S\right] + k_{m}} + k_{cat}^{u}\left[E_{u}\right]\frac{\left[S\right]}{\left[S\right] + k_{m}}\label{Eq:ptmMM1}
\end{equation}

In general, we will not have measured both the modified and unmodified protein, but rather will have measured the abundance of a covalently-modified peptide, $\left[E^{p}_{c}\right]$, and the relative abundance of the whole protein ($\left[E^{p}_{t}\right]$: unmodified plus modified). With this input data, we can not know whether the maximum fraction of covalent modified peptides is 2\% or 50\% of the total protein. To deal with this unknown, the maximum labelling fraction, $\phi_{max} \in [0, 1]$ must be estimated concurrently with kinetic parameters that post-translational modifications are posited to effect (\hyperref[Eq:ptmMM2]{Equation \ref{Eq:ptmMM2}}).

\begin{align}
c^{*} &= \underset{c\in C}{argmax} \sfrac{\left[E^{p}_{c}\right]}{\left[E^{t}_{c}\right]}\notag\\
\phi_{c} &= \phi_{max}\left(\frac{\sfrac{\left[E^{p}_{c}\right]}{\left[E^{t}_{c}\right]}}{\sfrac{\left[E^{p}_{c^*}\right]}{\left[E^{t}_{c^*}\right]}}\right)\notag\\
\nu_c &= k_{cat}^{p}\left[E_c^{p}\right]\frac{\left[S_c\right]}{\left[S_c\right] + k_{m}} + k_{cat}^{t}(1-\phi_c)E_{c}^{t}\frac{\left[S_c\right]}{\left[S_c\right] + k_{m}}\notag\\
 &= \left(k_{cat}^{p}\left[E_c^{p}\right] + k_{cat}^{t}(1-\phi_c)E_{c}^{t}\right) \frac{\left[S_c\right]}{\left[S_c\right] + k_{m}}\label{Eq:ptmMM2}
\end{align}

\subsubsection{Is the ``well-mixed'' assumption appropriate?}

Biochemistry can be greatly simplified when we assume that processes are spatially uniform and thus ``well-mixed''. Assuming that all cells within a culture are effectively equivalent is often an obligate requirement to allow researchers to collect enough physical material to quantify biological species. Extensive culture-wide information can then be abstracted back the content of a single cell. The use of many cells decreases the noise inherent in individual cells, but removes information about how cells meaningfully differ. 

In extreme cases, culture-wide behavior may not reflect the behavior of individual cells. As an example, consider a metabolic reaction: A + B $\rightarrow$ C. If the metabolite A is in present in only half of cells, and B is present in the complementary half, C will not be produced in either cell population. However, applying the culture-wide well-mixed assumption interaction between A and B would be possible, allowing the pathological synthesis of C to proceed. An absence of \textit{in vivo} colocalization can lead to the same fallacy when either cellular compartmentalization is ignored, or when enzymes \&/or metabolites are erroneously assumed to be evenly distributed within a compartment.

In some cases the assumption of cell-level similarity is likely appropriate at least to a first approximation. The best-case scenario for such an assumption to hold is likely during chemostat growth, were many generations of similar environmental conditions minimize how environmental heterogeneity and historical contingency affect growth. The culture-wide reproducibility of chemostats suggests the biological variance between individual cells is minimal, while it is also routine to verify the population is morphologically similar, at least in terms of cellular volume, during chemostat culture. While chemostat growth may be a best-case scenario for minimizing cell-to-cell variation, both random \cite{BarEven:2006dz, Kaern:2005gr} and non-random variation at the single cell level may be important at the population-level. The clearest example of a process which varies between cells is the temporal evolution of the mitotic cell-cycle whose sinusoidal oscillations could not be summarized by a bulk-sum analysis \cite{Hartwell:1974uy, Spellman:1998wj}. Such temporally-variably processes are not readily observed unless either cellular population are coordinated\cite{Hartwell:1974uy, Tu:2006cl}, or single cells can be tracked through morphology \cite{Herskowitz:1988ut}, microscopy \cite{Venturelli:2015ec} or single-cell sequencing \cite{Patel:2014dt}. Each of these latter techniques has revealed that otherwise similar cellular populations are frequently composed of distinct clusters of cells, but scope of such heterogeneity remains unclear. If single-cell concentrations of reaction species were available, SIMMER could be applied to model flux heterogeneity, however gaining the necessary input data is not currently possible. While advances in microscopy and automation are enabling the simultaneous tracking of larger numbers of proteins \cite{Ghaemmaghami:2003ds, Dubuis:2013cw}, understanding of single cell metabolic heterogeneity is currently limited by inadequate methods to detect metabolite levels of single cells \cite{Zenobi:2013il}.

As previously discussed, it is routine to measure the culture-wide abundance of biological species without accounting for inter-cellular variability. In general, such measurements are also carried out without regard to intracellular heterogeneity, either at the scale of compartments or even finer-scale localization. Eukaryotic cells are composed of distinct compartments with their own complement of metabolites and proteins which, jointly govern the compartment's metabolic activity. Within each compartment, metabolic activity may be further impacted by colocalization of enzymes that allow for efficient metabolic channeling of metabolites between sequential enzymatic steps \cite{Ovadi:1995wy}. To appropriately account for intracellular heterogeneity, metabolic species should be measured at their \textit{in vivo} concentration. Perhaps intracompartmental variable concentrations could be systematically incorporated into genome-scale models through appropriate microscopy experiments, but such concerns are secondary to accounting for compartment-wise metabolic variation.

The absence of compartment-scale resolution to metabolic data does not necessary greatly impact SIMMER. If we consider the concentration of a metabolite which is only found in a single compartment, and this compartment's volume is proportional to intracellular volume, the metabolite's intracellular concentration would be proportional to an absolute abundance measurement. This would result in inferred kinetic constant being off by a multiplicative constant, however the \textit{in vivo} reaction kinetics could still be determined. If however, metabolites are differentially partitioned across conditions (between the cytosol, mitochondria and vacuole in particular), relative concentrations in compartments will not reflect that of whole-cells \cite{Kitamoto:1988wc}. Differential partitioning could explain why phenylpyruvate was erroneously predicted as an inhibitor of DAHP synthase rather than the true regulator phenylalanine \cite{Schnappauf:1998ec}. Phenylalanine is partially stored in the vacuole, while phenylpyruvate is likely primarily cytosolic \cite{Kitamoto:1988wc} .  Because phenylalanine is made from phenylpyruvate in the cytosol, phenylpyurvate may be a better proxy for cytosolic phenylalanine concentrations than whole-cell measurements. In addition to the vacuole, estimating the concentrations of mitochondrial metabolites is also challenging. This is because relative mitochondrial volume may vary across conditions, as suggested by the strong covariation of mitochondrial enzymes (Supplemental Figure 2) and because many mitochondrial metabolites are also present in the cytosol but do not readily mix.  

To better deal with the clear problems apparent when compartmentalization is ignored, in eukaryotes, each major metabolic compartments should be treated with the same care as separate experimental conditions. Techniques are emerging that allow for analysis of both metabolomics and proteomics of individual sub-cellular compartments \cite{Klie:2011kq, Wuhr:2014fr}. Because these methods are relatively gentle, it is unclear whether such approaches could be used to quantify the mitochondrial metabolome due to its great activity, such methods may be necessary to kinetically model the yeast TCA cycle. A clearer investigation of compartment-specific biomolecules should be complemented by microcopy-based approaches to measure compartment volumes \cite{JENSEN:1993bz, Ghaemmaghami:2003ds}.

\subsubsection{Evaluating the veracity of modeling assumptions}

While I have discussed the numerous ways in which culture-wide measurements of reaction species and modeling assumptions may limit our ability to accurately determine reaction kinetics, the limitation on how to address these caveats is primarily experimental rather than conceptual. Measurements of post-translational modification and and compartment-specific quantification of metabolite abundances could be readily incorporated into SIMMER and would even allow for the testing of their significance. Insufficient data greatly limits the number of reactions whose kinetics can be interrogated, while for other reactions our inability inability to describe their reaction kinetics despite measuring the primary reaction species  and possible regulators suggests that other factors are kinetically important. While missing data could limit our ability to identify appropriate reaction kinetics or could even lead to false-positive predictions of regulation, our ability to accurately predict reaction flux through numerous reactions using only reaction species and regulators suggests that our simplified models are more likely appropriate than not.

From my analysis for \textcolor{red}{n reactions with ostensibly accurate data, the kinetics of n reactions}, simple Michaelis-Menten kinetics including at most two regulators can accurately relate metabolite and enzyme concentrations to reaction flux. This is impressive firstly due to the large number of reasons why this could fail; but also because mechanistic modeling of the kinetics of well-studied reactions rarely reduces to Michaelis-Menten kinetics, and often many regulators are thought to collectively regulate activity \cite{Hill:1977vm}.  Under assay conditions, significant deviations from Michaelis-Menten kinetics may exist and the activity of regulators can be demonstrated; for the purpose of understanding flux control, a reaction form does not need to be principled mechanistically, but rather must only account for an approximately correct relationship between species and resulting flux across the physiological conditions investigated \cite{Fell:1997wg}. 

When trying to estimate the kinetics of reactions which cannot be described using metabolites, enzymes and flux, additional experimental data will surely shed light on the role of unaccounted for sources of regulation, however in other cases inferring appropriate reaction kinetics may require alternative modeling assumptions rather than additional data. Such modeling assumptions could be easily relieved, as in principle, using SIMMER any relationship between reaction species and resultant flux can be evaluated using flux. If an alternative reaction mechanism could be significantly supported over alternative contenders this would suggest its validity in line analogously to how reaction mechanisms are test \textit{in vitro} \textcolor{red}{cite}. Other kinetic phenomenon such as cooperativity \textcolor{red}{cite} could also be readily tested, as was demonstrated when testing whether cooperativity of allosteric activators or inhibitors was quantitatively supported.

\subsubsection{Application of SLIMER using more comprehensive data, broader conditions and new organisms}

Because we have characterized a reproducible set of conditions that contain much of the meaningful metabolic variation in yeast, I hope this dataset will serve as an important scaffold for future work to build upon.  In order to better understand yeast metabolism it will be important to refine and expand our input data in the ways that I have discussed above, and it will also be important to expand the number of experimental conditions used to study kinetics. Investigating additional conditions is important for two reasons. First, when too few conditions have been tested, alternative models of reaction kinetics may not be discriminated, for instance due to correlation of putative regulators. Second, we can only identify important regulation that impacts the conditions that we have studied. By expanding the number of conditions that are investigated, we can unmask regulation that is important in these conditions, and our approach can identify this regulation by virtue of the kinetic importance it gains in these new conditions

To improve kinetic inference, when choosing additional conditions to study, ideally, for each reaction, we want to maximize the number of distinct input-output pairs that can found (i.e. combinations of flux and all potential reaction species) and prioritize conditions spanning meaningful metabolic transitions. These two goals are broadly compatible, as diverse conditions will generate extensive multi-`omic variation and will reveal metabolic regulation which is necessary to operate at these states. For example, if we were to grow yeast using ethanol as a carbon source rather than using glucose, this would result in a unique pattern of flux, enzymes, metabolites and the appearance of conditionally-meaningful regulation. This would allow us to confirm the regulation which facilitates the glycolytic to gluconeogenic switch, but also would likely inform regulation more broadly due to the addition of new metabolic state departed from previously charactered conditions \cite{Zampar:2013fr}. Similar logic could be used to better understand the specific metabolic impact of regulators of protein abundance, be they important transcription factors such as Gcn4 or signaling pathways such as Ras.  For example, a hypoactive variant of the Ras-regulator Ira2 in yeast greatly impacts transcript and protein abundance, but also alters the levels of glycolytic metabolites and flux \cite{Breunig:2014bu}. Through SIMMER's connection of $\left[regulator \rightarrow enzymes \rightarrow metabolites \rightarrow flux\right]$, the specific manner in which hypoactive Ira2 impacts flux could be discerned from ancillary changes in enzyme and metabolite. 

While the approach of SIMMER is a powerful way to understand yeast metabolism, its application to a well-studied, relatively simple, organism does not fully highlight SIMMER's utility. While the metabolic regulation of yeast has been extremely well studied through decades of \textit{in vitro} biochemistry and genetic, such time-intensive characterization is not feasible for the majority of organisms.  While previously reported \textit{in vitro} regulators can be prioritized and physiologically validated using SIMMER, in general such prior assumptions are unnecessary or can be supplanted by regulation favored through genetic conservation. When applying SIMMER to new organisms, or reactions with little or no previous chemical interrogation, the use of hypothetical regulators will be a particularly powerful way to prioritize regulatory metabolites. The reductionist approach of SIMMER will also be greatly useful when investigating the metabolic regulation of more complicated organisms including humans. In higher eukaryotes, flux inference based on steady-state assumptions and parsimony largely break down, and these methods are largely supplanted by pathway-specific methods based on the dynamic turnover of isotopically-labelled metabolites. The scalability of SIMMER could help to understand the kinetics of these pathways, as measurements of flux are routinely complimented with metabolomics and the generation of mutants which perturb pathway activity and metabolites.

\subsubsection{Translating reaction-level kinetics into metabolism-level predictions}

Determining reaction kinetics is important because it tells us how each reaction in a complex dynamical system will behave a function of its inputs. This is of massive importance in metabolic engineering because if we want to flux through a pathway of interest which produces a commodity chemical, then ideally we would like to know how pathway flux will respond to changes in pathway enzymes. Due to the difficulty of reaction form inference, current models of metabolic regulation are limited by an incomplete understanding of how reaction species. Such models may lead to predicted metabolite-enzyme-flux relationships which agree with the data they are fitted on, however they rarely generalize when applied to new conditions. Such naive models may suggest how flux will respond to genetic perturbations, however their inaccuracies will make it is far more difficult to understand how the metabolic network (and indeed broader regulation which will be touched on below) will respond to such a perturbation. 

Due to these limitations, progress in metabolic engineering has generally been born out of brute force trial and error rather than principled application of metabolic model predictions. Understanding how metabolism responds to changes in enzyme abundance is well-studied in Metabolic Control Analysis (MCA), however as MCA generally requires a complete kinetic description of all relevant reactions, to date, its application has been largely impractical. SIMMER is an ideal tool to fill this void by providing the means to identify reaction forms which are kinetically appropriate across a wide-range of conditions. To demonstrate the application of SIMMER to MCA, I generated a MCA-based model of glycolysis which reproduces two primary aspects of yeast glycolytic control. Similar methods can be used to predict changes in flux in response to genetic perturbations or the removal of allostery. This would entail introducing an instantaneous change in a parameter (enzyme abundance or removing allostery) and then following the time-evolution of the system as it reaches a new steady-state. Because each reactions form is fitted across a wide-range of metabolic conditions and contain relevant regulation, the behavior of individual steps should accurately respond to moderate changes, resulting in reasonably accurate predictions of resulting pathway flux.


\subsection{Towards a Comprehensive Model of Protein Expression}

By analyzing systematic deviations between proteins and transcripts during Chapter \ref{ch:pt_compare} I implicated post-transcriptional regulation as providing a highly relevant layer of protein-expression control. This analysis was not ideal due to differences in experimental design between the two compared datasets. By measuring proteins and transcripts in the same culture and ideally accounting for biological variance through replication, a more faithful comparison between these species could be conducted.  Such an analysis could also disambiguate the role of regulation of translational efficiency as opposed to protein degradation through experimental determination of one (or both) of these processes using existing methods \textcolor{red}{cites}.

The central dogma of molecular biology revealed to researchers that information in a cell largely flows from DNA to RNA to protein. The limitation of this nearly-universal relationship is that it is non-quantitative, while the composition of the proteome is governed by the variable rate of information transfer both both between-gene and across-conditions. Towards this end, a quantitative central dogma of molecular biology is needed to account for how changes in protein expression are related to the intra- and extra-cellular environment.

One application of such a quantitative models of protein expression is that models of expression could be directly tied into models depending on protein activity, such as metabolism.  One essential challenge when trying to predict the behavior of metabolism is that an induced change in protein expression will result a change in flux and metabolite abundance which will result in a secondary change in gene expression. This change could be due to the cost of overexpression \textcolor{red}{cites}, impacts on nutrient sensing on transcription or translation \textcolor{red}{cites}, or other effects such as responses to osmolarity or pH \textcolor{red}{cites}. When gene expression responses to a genetic or environmental perturbation are unknown, the accuracy of metabolic prediction will decline as metabolism is pushed beyond measured conditions.

If an integrated expression-metabolism model was possible, metabolic behavior could be predicted as a function of the environment and expression; while expression could be predicted based on the environmental and metabolic state. The industrial impacts of this development would be massive; as by reliably connecting the impact of genetic and environmental changes to their resulting metabolic impact, pathway fluxes could be efficiently altered to produce either native compounds or favor synthetic pathways which synthesize foreign compounds. In this case, not only would pathway flux operate as a tunable knob, but the flux into a desired pathway could be balanced relative to an organism's biosynthetic requirements in order to achieve predictable steady-state synthesis.

The utility of expression-metabolism models for unlocking the industrial capabilities of microbes are that metabolic behavior can be predicted based on the organism's genetic and environmental state. Establishing this link between genotype and phenotype, through context dependent mechanistic models, is the same essential connection that is sought when investigating the genetic basis of complex traits and disease. Expanding this connection to higher-level traits could be accomplished using the same principles as those used to dissect the complexity of yeast. When studying disease, first models of cellular behavior need to be improved; then related to organ-level behavior and ultimately these systems must be integrated under that auspices of medicine, itself an endeavor which has worked to generate \textit{ad hoc} models of disease occurence. 

further levels of causality