It seems like science fiction to suggest that, the gross complexity of an organism as complicated as \textit{homo sapiens} could be reduced to a model of physiology that is sufficient to fully understand how genetic variation and environmental exposures shape phenotypic variation at every scale. However, I firmly believe that the field is poised for just such a model to be created for a suitable microbe. Such a model would required detailed experimental measurements of nearly all cellular species, a technical requirement which is becoming increasingly more feasible through the ever increasing accuracy and breadth of sequencing, mass spectrometry and high-throughput microscopy. These measurements alone can characterize the static state of an organism, but they can not tell us about the relationships between chemical species which lead to the emergent property of physiology. 

To gain insights into how species interact, we must note how they covary. Generating such variation can be done through a combination of genetic and environmental perturbations which would ideally generate as much orthogonal physiological variation as possible. Regardless of how many conditions could be practically assessed, the large space of possible interactions between variables would challenge analysis. To guide the create of models, core principles of molecular biology and biochemistry can serve as a scaffold to lay biological information. I believe that my thesis research espouses this philosophy, provides clear practical examples of how experimental data can be used to guide bottom-up reconstruction of complex systems.

During Chapter \ref{ch-quant_anal}, I discussed practical considerations when using mass spectrometry data.  While mass spectrometry is an ideal platform for quantifying important physiological variables, such as metabolites, peptides/proteins and lipids, in some cases experimental data does not agree with the assumption of log-normal variability. When analyzing variation in a single class of species, such as metabolites, deviations can be dealt with through non-parametric approaches, but when a dataset is meant to be integrate it into systems-level models, summarizing data based on parametric assumptions, is of great utility.  This is because, when creating models, it is important to know how the variation in model outputs in related to variation in inputs. Attempting to unlock the power of parametric statistics mass spectrometry I noted that mass spectrometry datasets variably agree with the log-normal assumption, and discussed the use of more flexible heteroscedastic models of parametric variation which may be important in some cases.

As discussed during Chapter \ref{ch-simmer}, addressing the metabolism of yeast, one of the most complicated properties of a cell, I was able to reduce a non-linear system of differential equations, to a set of possible reaction-level models which could be efficiently solved piece-meal. The power of this approach is currently limited by an incomplete characterization of relevant species which prevents an exhaustive analysis of all relevant reactions and all possible models of their kinetics. Despite these current limitations, the approach of SIMMER will remain computationally tractable at scale, and when additional conditions are assessed, this method could practically quantitatively differentiate a true kinetic model to the extent that alternative models can be statistically distinguished by virtue of their variation. While inference in SIMMER proceeds in a reaction-by-reaction manner, inferred kinetic models are sufficient to describe how components interact and ultimately establish pathway or metabolism-level behavior. As such, with the appropriate addition of further experimental data, SIMMER will be an ideal platform for predicting how pathway fluxes change in response to nutrient inputs or alterations of protein expression.

During Chapter \ref{ch-pt_compare}, I discussed another important system, how variable levels of proteins are produced across diverse growth conditions. Studying each gene independently, I determined whether nutrient induced changes in protein abundance can be explained based on changes in their cognate transcript, when departures between proteins and transcripts may be due to post-transcriptional regulation, and when these departures are likely due to experimental noise. This analysis showed that while strong transcriptional changes propagate into corresponding changes in protein abundance, additional layers of post-transcriptional control are relevant.

The conceptual ideas presented 