\chapter{Multiple Sources of Regulation Impact Nutrient-Dependent Protein Expression in Yeast\label{ch:pt_compare}}

\section{Introduction}

Each protein's expression is the bulk sum of its regulation through transcription, mRNA stability, translation and protein stability. These properties are tuned over evolutionary time-scales to accomplish massive variable expression of genes \cite{Csardi:2015kx}, and to refine how each gene interacts with regulatory components.  The source of these overall between-gene differences is likely variable sequence composition which alters, for instance, promoter strength or ribosome affinity independently of a cell's physiological state. On top of these relatively static constraints on expression, an organism must physiologically tune protein expression through dynamic changes in transcription, translation and stability.  While static between-gene and dynamic between-condition differences in expression involve the same regulatory components, there is little reason to suspect that these scenarios will be equivalent in terms of the relative importance of each transcription, translation and stability in controlling gene expression.  For example, transcriptional regulation means very different things when we consider between-gene versus between-condition regulation.  Between-gene differences in RNA expression are primarily due to promoter strength, while differences in transcription of a given gene across conditions are primarily governed by transcription factor binding.  

%For example, an unnecessarily labile protein would be energetically wasteful and thus maladaptive, but if this protein's stability changes in response to cellular state, then the cell can change its state faster than dilution would allow 
 
To quantitatively understand how protein levels are accomplished, we can mathematically represent changes in transcript and protein levels of any gene \textit{i} in a given condition \textit{j} using a pair of coupled differential equations (Equation \ref{Tdiffeq} and  \ref{Pdiffeq}) that account for the role of transcription ($\gamma$), RNA degradation ($\delta$), translation ($\alpha$), protein degradation ($\lambda$), and dilution due to growth ($GR$).  

\begin{subequations}
\begin{align}
\frac{dT_{ij}}{dt} &= \gamma_{ij} - T_{ij}\delta_{ij} - T_{ij}GR_{j}\label{Tdiffeq}\\
\frac{dP_{ij}}{dt} &= \alpha_{ij}T_{ij} - P_{ij}\lambda_{ij} - P_{ij}GR_{j}\label{Pdiffeq}
\end{align}
\end{subequations}

Using this formalism, static differences in regulation exist if a parameter varies across genes ($i \in 1 ... I$), while dynamic differences in regulation exist if a parameter varies across condition ($j \in 1....J$) for a given gene.  In either case, meaningful regulation exists if the rate of a process varies greatly across genes or conditions, and this change causally results in a significant change in protein levels. While these equation provide notational convenience, determining the relative regulatory importance of each process is difficult because generally, protein levels are an integral over past transcript abundance and regulatory states (Equation \ref{dynamic_expression}), rather than being solely dependent on a single state.

\begin{align}
P_{T} &= P_{0} + \int_{t = 0}^{T} \alpha_{t} T_{t} - (\lambda_{t} + GR_{t}) P_{t} \hspace{2mm} dt \label{dynamic_expression}
\end{align}

While this relationship is complicated, Jovanovic et al. 2015 demonstrated that by determining how regulatory parameters vary across a dynamic transition using extensive experimentation and dense temporal sampling, the relative role of transcription, RNA stability, translation and protein stability could be assessed on a per-gene basis \cite{Jovanovic:2015hp}. The downside of this approach is that extensive experimentation is necessary to interrogate a single focused physiological transition which may be unrepresentative of general regulation.

As an alternative to such detailed interrogation of gene expression dynamics, we and others \cite{Belle:2006hv, Csardi:2015kx}, noted that at steady-state, protein abundance can be expressed relative to steady-state transcript abundance and time-invariant parameters, allowing a single condition to be characterized by measurements at a single time-point.  From equation \ref{Pdiffeq}, we can see that if we are only interested in how protein levels are established, transcript abundance ($T_{ij}$) is sufficient to account for the combined influence of transcription, RNA degradation and transcript dilution. In this case, changes in protein abundance due to the combined influence of variable transcript abundance, translation and protein degradation rate still follows equation \ref{Pdiffeq} but this relationship can be expressed as a time-invariant function of these variables (Equation \ref{steady_expression}).

\begin{align}
&\frac{dP_{ij}}{dt} = \alpha_{ij}T_{ij} - P_{ij}\lambda_{ij} - P_{ij}GR_{j}\notag\\
&\frac{dP_{ij}}{dt} = 0 ; \frac{dT_{ij}}{dt} = 0 \notag\\
&\dot{P}_{ij} = \frac{\dot{\alpha}_{ij}\dot{T}_{ij}}{\left(\dot{\lambda}_{ij} + \dot{GR}_{j}\right)}\label{steady_expression}
\end{align}


Building off of this approach which has previously been used to characterize between-gene differences in regulation, here we report a quantitative comparison of protein and mRNA relative abundance across 25 disparate \textit{S. cerevisiae} steady-states.  On a per-gene basis we determined whether protein expression follows from nutrient-dependent changes in mRNA expression or whether meaningful differences between protein and mRNA expression exist.  Based on data from 1161 genes, we find that while transcriptional changes do propagate into changes in protein abundance, additional layers of control either due to regulation of translational efficiency or protein stability are also important. As these growth conditions span diverse physiological and metabolic challenges, control of expression across these conditions should be generally representative for yeast.

\section{Results}

Our interest in how the overall expression of proteins, and to a larger extent all of physiology, in yeast is shaped by the interaction of the environment and genetically hard-wired regulation follows from previous characterization of how environmental variation impacts transcripts, metabolites and most recently proteins (Chapter \ref{ch:simmer}) and metabolism \cite{Brauer:2008jn, Boer:2010fb}. In each study, 25-36 steady-states were generated by simultaneously varying both culture growth rate and which nutrient limits growth (i.e. a \underline{C}arbon, \underline{N}itrogen, \underline{P}hosphorous source, as well as \underline{L}eucine and \underline{U}racil in appropriate auxotrophs). As discussed in Chapter \ref{ch:simmer}, it is clear that transcripts, proteins and metabolites are differentially impacted by growth rate and limiting nutrient; with transcript abundances being strongly impacted by growth rate, metabolites primarily reflecting limiting nutrient and proteins showing an intermediate response.

While measured transcript and protein levels \cite{Brauer:2008jn}\textcolor{red}{+ Hackett} are not totally equivalent in terms of experimental design and used distinct biological cultures as will later be discussed, the union of these two datasets represents the most diverse collection of paired transcript and protein data to date, warranting a conservative, exploratory analysis.  Each study focused on quantifying protein and transcript levels relative to a shared reference condition \cite{Quackenbush:2002kl}.  Since most technical variability will similarly impact both the experimental sample and reference, this approach is ideal for quantifying the relative abundance of proteins and transcripts. Treating both transcript relative abundance and protein relative abundance as lognormal \cite{Quackenbush:2002kl, Cox:2008ir}, log-protein abundance can be expressed as a linear function of log-transcript abundance and the combined influence of post-transcriptional regulation and dilution by growth (Equation \ref{across_cond_steady_expression}).

\begin{align}
&\sfrac{\dot{P}_{ij}}{\dot{P}^{ref}_{i}} = \sfrac{\dot{T}_{ij}}{\dot{T}^{ref}_{i}}\frac{\dot{\alpha}_{ij}}{\left(\dot{\lambda}_{ij} + \dot{GR}_{j}\right)}\notag\\
&\dot{P}_{ij} = \dot{T}_{ij}\frac{\dot{\alpha}_{ij}}{\left(\dot{\lambda}_{ij} + \dot{GR}_{j}\right)}\sfrac{\dot{P}^{ref}_{i}}{\dot{T}^{ref}_{i}}\notag\\
&log_{2}\dot{P}_{ij} = log_{2}\dot{T}_{ij} + \log_{2}\frac{\dot{\alpha}_{ij}}{\dot{\lambda}_{ij} + \dot{GR}_{j}} + C_{i}\label{across_cond_steady_expression}
\end{align}

Based on equation \ref{across_cond_steady_expression}, we first determined the extent to which steady-state protein abundances reflect steady-state transcript abundances assuming no post-transcriptional regulation (i.e. $\alpha_{iJ}$ and $\lambda_{iJ}$ can vary across genes, but are constant across samples).  Across all genes and conditions, changes in protein abundance are only weakly associated with changes in protein abundance (r$^{2}$ = 0.131), although stronger transcriptional changes are more faithfully transmitted into meaningful changes in protein abundance (Figure \ref{ptscatter}A).  An increase in transcript-protein association for genes under strong transcriptional control, suggests that the poor correlation between proteins and transcripts may be variable across genes and this consistency may be better studied on a gene-by-gene basis.  When comparing the correlation of proteins and transcripts across the 25 experimental conditions, protein and transcripts are significantly positively correlated for only 59\% of genes; for the remaining genes, protein and transcript abundance are effectively uncorrelated (40\%) or even significantly anti-correlated ($\sim$1\%) (Figure \ref{ptscatter}B).

\begin{figure}[h!]
\begin{center}
\includegraphics[width=0.8\textwidth]{ch-pt_compare/Figures/PTcompare1.pdf}
\caption[Correlation of protein and transcript relative abundance]{Correlation of protein and transcript relative abundance.  \textbf{A)} Protein and transcript abundance relative to the gene-average are compared across every gene and condition using a hexbin plot (i.e. bivariate histogram).  Four contours are shown where an equal density of protein-transcript relative abundance pairs exist.  \textbf{B)} The by-gene spearman correlation between protein and transcript relative abundance for every gene is shown.  Genes are colored based on whether protein and transcript relative abundances are significantly correlated (green), uncorrelated (gray) or anticorrelated (red) at an FDR of 0.05.}
\label{ptscatter}
\end{center}
\end{figure}

For more than 40\% of highly expressed yeast genes, protein and RNA expression are effectively uncorrelated.  This gross disconnect could either be due to noise/irreproducibility or could be due to meaningful differences in proteins-per-transcript. Differences in proteins-per-transcript could be due to variable growth rate across cultures or differential regulation of translational efficiency or protein degradation (Equation \ref{across_cond_steady_expression}).  Growth rate can be largely excluded from this list as, accounting for variable growth rate across conditions does not meaningfully improve the correlation between protein and transcript abundance changes (r$^{2}$ = 0.152).  To determine whether noise or post-transcriptional regulation is primarily responsible for the low correlation between transcript and protein abundance changes, potential regulation through either regulation of translational efficiency ($\alpha$) or degradation rate ($\lambda$) can be isolated by looking at variation in proteins-per-transcript across conditions (Equation \ref{eq:p_per_t})

\begin{align}
&log_{2}\dot{P}_{ij} = log_{2}\dot{T}_{ij} + \log_{2}\frac{\dot{\alpha}_{ij}}{\dot{\lambda}_{ij} + \dot{GR}_{j}} + C\notag\\
&log_{2}\left(\frac{\dot{P}_{ij}}{\dot{T}_{ij}}\right) \propto \log_{2}\frac{\dot{\alpha}_{ij}}{\dot{\lambda}_{ij} + \dot{GR}_{j}}\notag\\
\label{eq:p_per_t}
\end{align}

When investigating variation in proteins-per-transcript, we can consider that meaningful regulation will likely impact multiple genes in a similar manner and furthermore differential regulation will primarily track the experimental design (i.e. regulation influenced by limiting nutrient and/or growth rate).  In contrast, most possible sources of noise introduced from noise in the proteomics and transcriptomics data will either be independent across genes (i.e. technical variance) or will covary across genes but not in a manner meaningfully related to the biological design of the experiment (i.e. due to culture irreproducibility or biological variance) \cite{Leek:2007kn}.

Visualizing shared variation in transcript and protein abundance as well as proteins-per-transcript abundance across genes (Figure \ref{ptHM}A) we see that proteins-per-transcript strongly covaries across genes.  Furthermore, similarly to transcript and protein abundance, this covariation is strongly governed by growth rate and limiting nutrient, suggesting that biologically meaningful differences in proteins-per-transcript exist (Figure \ref{ptHM}B). The major post-transcriptional regulatory factors driving shared variation in proteins-per-transcript can be accounted for by summarizing these latent factors using principle components analysis and then determine which factors effect individual genes using Jackstraw \cite{Chung:2015bq} (Figure \ref{ptHM}C). Treating the top four principle components of the proteins-per-transcript matrix as a surrogate for these latent effects, substantially improves the correlation between protein and the shared effect of transcript abundance and fitted proteins-per-transcript (r$^{2}$ = 0.446).

\begin{figure}[h!]
\begin{center}
\includegraphics[width=0.8\textwidth]{ch-pt_compare/Figures/PTcompare.pdf}
\caption[Protein and transcript relative abundance and implied proteins-per-transcript are each strongly influenced by growth rate and limiting nutrient]{Protein and transcript relative abundance and implied proteins-per-transcript are each strongly influenced by growth rate and limiting nutrient. \textbf{A)} Heatmap summaries of transcript and protein relative abundance as well as proteins-per-transcript ($log_{2}\left[protein\right] - log_{2}\left[mRNA\right]$) is shown. The organization of rows in each heatmap is the same and was determined through hierarchical clustering of the proteins-per-transcript matrix using pearson correlation with average linkage. \textbf{B)} The principal components reflecting systematic variation in each dataset were found and the percent of total dataset variability explained by each significant principal component is shown. \textbf{C)} A lower dimensional representation of the raw proteins-per-transcript can be generated by determining which genes are influenced by each principal component.}
\label{ptHM}
\end{center}
\end{figure}

By analyzing proteins-per-transcript, the combined influence of variable translational efficiency, protein degradation and growth can be quantified, however specifically implicating variable translational efficiency or protein degradation as the causal factor driving the departures between proteins and transcripts is difficult. To get a better handle on how proteins-per-transcript is meaningfully regulated across these conditions and which genes are the focus of this regulation, we focused on the large differences in proteins-per-transcript that are apparent at nitrogen-limited slow growth. Accordingly, we used gene set enrichment analysis (GSEA) to determine whether any gene ontology (GO) categories showed systematically elevated or depressed proteins-per-transcript \cite{Subramanian:2005jt}. To deal with several general challenges in GSEA including GO categories which are largely overlapping, nested and differing greatly in size, we used GSEAMA, an approach based on LASSO regression \cite{Tibshirani:1996wb} to identify GO categories associated with proteins-per-transcript. To visualize how these GO categories are grossly associated with elevated or depleted proteins-per-transcript in nitrogen limitation, the approach of voronoi tesselation \cite{Otto:2010br} was built upon by using GO category significance to guide the hierarchical layout of genes. This results in a layout of genes which organizes the major gene-set dependent variation to show its overall impact.

Using GSEAMA, we determined which biological processes GO categories are best able to predict proteins-per-transcript and these gene sets were used to guide organization of genes into a hierarchical voronoi tesselation (Figure \textcolor{red}{figure}) \cite{Halligan:2007ds}.  Using this approach, it is clear that ribosomes and metabolic enzymes are differentially regulated in nitrogen limitation, with fewer ribosomes being produced per-transcript and an elevated number of enzymes per-transcript. To more globally determine how gene function is linked to proteins-per-transcript, genes were clustered into 20 groups based on pearson correlation.  To identify classes of genes enriched in these clusters, iPAGE was used to assess gene set enrichment, revealing 8 meaningful associations with gene ontology categories (Figure \ref{ch-pta:fire_page}A) \ref{Goodarzi:2009cf}. To test whether common amino acid motifs were linked to the gene clusters, motifs that are significantly enriched or depleted in specific clusters were identified Fire-pro, revealing 11 amino acid motifs whose frequency varies greatly across the 20 clusters (Figure \ref{ch-pta:fire_page}B) \cite{Lieber:2010fr}.

% These two categories are particularly interesting as ribosomes are among the most stable proteins when they are actively translating and the least-stable when translation is inhibited \cite{Belle:2006hv}. Since amino acid concentrations are low in nitrogen limitation, ribosomes will tend to stall, fall apart and be degraded.  In contrast, metabolic enzymes are prioritized during nitrogen limitation, likely because they are needed to restore an appropriate level of amino acids.  The source of this control is unclear, although it is intriguing proposition that metabolites could stabilize proteins or increase translation.  Such an interaction between the metabolome and proteome could explain why metabolic enzyme concentrations more strongly reflect the cellular nutritional environment than their corresponding transcripts. 

\begin{figure}[h!]
\begin{center}
\subfloat{
\includegraphics[width=0.47\textwidth]{ch-pt_compare/Figures/PAGE_rawTE_cleanup.pdf}
}
\hspace{1mm}
\subfloat{
\includegraphics[width=0.47\textwidth]{ch-pt_compare/Figures/FIRE-pro_rawTE_cleanup.pdf}
}
\caption[Characterizing clusters of genes with similar variation in proteins-per-transcripts across conditions]{Characterizing clusters of genes with similar variation in proteins-per-transcripts across conditions. \textbf{A)} 
iPAGE and FIRE-pro}
\label{ch-pta:fire_page}
\end{center}
\end{figure}


\section{Discussion}

As would be assumed from the nature of transcriptional control, changes in transcript levels generally propagate into corresponding changes in protein level.  Noise, both in estimating transcript and protein abundance partially obscures the association between variables, a factor which would diminish the correlation between protein and transcript levels. Addressing this diffusion has been previously suggested as a way to reconcile the poor associations between protein and transcript abundances across genes \cite{Csardi:2015kx}.  While uncertainty certainly diminishes the association between individual transcripts and proteins, noise will in general affect different genes independently or in a manner associated with unmeasured or technical covariates rather than biological ones \cite{Leek:2007kn}. Instead, we see that the number of proteins-per-transcript is greatly influenced by the design of our experiment, strongly tracking both growth rate and which nutrient is limiting growth.

The existence of meaningful post-transcriptional regulation across variable yeast nutrient environments could be expected based upon how transcripts, proteins, and metabolites \cite{Boer:2010fb} grossly respond to nutrient variation \ref{ch:simmer}. Comparing transcripts, proteins and metabolites, transcriptional control is more tightly coupled to the growth and stress state of the cell, while metabolite levels are profoundly shaped by nutrient limitation. Protein levels show greater growth-rate dependent variation than metabolites, but more nutrient-dependent variation than transcripts. As both translation and protein degradation are tightly coupled to the nutritional state of the cell \cite{Klumpp:2009ic}\textcolor{red}{degradation during nutrient stress}, post-transcriptional regulation serves as an additional layer of regulation where the metabolome can tune the primary transcriptional response.

Focusing on nitrogen-limited cultures where genes differed greatly in proteins-per-transcript, we noted that the genes which with a lower relative translational efficiency or a greater translational efficiency are strongly related to their function.  In particular, two of the largest classes of proteins in terms of absolute levels, ribosomes and metabolic enzymes were regulated in an opposite manner, with ribosomes being selectively degraded in nitrogen limitation and metabolic enzymes preserved.  This finding dovetails strongly with previous findings that ribosomes are among the most stable proteins in the cell during active translation \cite{Belle:2006hv}, but are actively degraded under conditions of amino acid limitation \cite{Natarajan:2001ke, Washburn:2003ff, Zundel:2009dy}; meanwhile metabolic enzymes were stable regardless of translational state \cite{Belle:2006hv}.  As amino acids are greatly limiting in slow-growth nitrogen limitation, ribosomes which are unable to find charge amino acids are regularly resorbed back into the amino acid pool of the cell \cite{Zundel:2009dy, Xu:2013do}.  This relationship, suggests that under physiological conditions, protein degradation is a major factor governing steady-state protein levels.

Insufficient nitrogen levels may play into the continually degradation of existing protein, but cells which differ greatly in standing concentrations of amino acids and ribosomes likely also differ greatly in the rate which proteins are synthesized. These ``global'' effects will similarly impact all genes in a given culture, but likely vary greatly between growth conditions \cite{Klumpp:2009ic}. It is less clearly whether variable translational efficiency could lead to the pronounced structure of Figure \ref{ptHM}C, although such regulation is likely based on the considerable variation in the translational efficiency of genes throughout the yeast meiosis, \cite{Brar:2012ig}. 

To the extent that the Brauer et al. 2008 and Hackett proteomics datasets focus on yeast growing under a set of similar growth conditions, these experiments are generally comparable, however there are several meaningful differences between these datasets that must be considered. One large difference between these studies is that while a CEN.PK-derived strains were used in Brauer et al. 2008 \cite{vanDijkenJP:2000er} while S288C-derived strains were used in the Hackett dataset \cite{Winston:1995io}. The divergence between these two strains, 0.13\% \cite{Schacherer:2007ck}, is modest but far less than the divergence of 0.6\% between the BY4716 and RM11-1a strains \cite{Foss:2007ej} which have been used to investigate the impact of segregating genetic variation on transcript and protein abundance \cite{Brem:2005gh, Foss:2007ej}. Genetic variation affecting transcript or protein abundance will not impact our inference as strain-specific effects would equivalently impact experimental and reference conditions. In contrast, strain x condition effects would add noise to the proteins-per-transcript signal or if multiple genes are affected (i.e. a hotspot), this effect could look similar to real post-transcriptional regulation. The results of Smith et al. 2008, suggest that even in a cross between the highly divergent BY4716 and RM11-1a strains, strain effects and nutrient effects are far stronger than strain x nutrient effects \cite{Smith:2008vy}. As in this study, there is less genetic variation between CEN.PK and S288C, and larger variation in nutrient environments, strain x condition should be even weaker.

The total variation in proteins-per-transcript across all genes and conditions ($\sigma^{2}_{P/T}$) arises from five meaningful sources of variation (Equation \ref{eq:sources_of_ptvar}). These components are, condition-dependent regulation ($\sigma^{2}_{D}$), genetic variation that differentially impacts transcripts/proteins across conditions ($\sigma^{2}_{GD}$), systematic differences in a subset of conditions between the two studies, such as differences in media glucose ($\sigma^{2}_{DB}$), biological irreproducibility ($\sigma^{2}_{B}$) and technical irreproducibility ($\sigma^{2}_{T}$).

\begin{equation}
\sigma^{2}_{P/T} = \sigma^{2}_{D} + \sigma^{2}_{GD} + \sigma^{2}_{DB} + \sigma^{2}_{B} + \sigma^{2}_{T}\label{eq:sources_of_ptvar}
\end{equation}

Three of these variance components ($\sigma^{2}_{D}$, $\sigma^{2}_{GD}$, $\&$ $\sigma^{2}_{DB}$) could result in the structured variation whose collective impact we estimate as accounting for $\sim45\%$ of variation in proteins-per-transcripts. Some fraction of this structured variation is due meaningful regulation across these conditions (transcriptional and post-transcriptional) and additional variance is due to systematic differences between these datasets ($\sigma^{2}_{GD}$, $\sigma^{2}_{DB}$).  Because the magnitude of $\sigma^{2}_{GD}$ and $\sigma^{2}_{DB}$ is unknown, we can place a lower limit on transcriptional regulation as $\sfrac{0.131}{0.446}$ as \textcolor{red}{28.1\%} but cannot say much more about the relative strength of transcriptional and post-transcriptional regulation.

While in this study we have suggested a meaningful role of post-transcriptional regulation in altering the yeast proteome across nutrient conditions, a further study is warranted to clarify the relative influence of transcript abundance, translational efficiency and protein degradation. Such as study would remove several sources of excess noise present in this dataset ($\sigma^{2}_{GD}$, $\sigma^{2}_{DB}$, $\&$ $\sigma^{2}_{B}$), likely greatly improving the association between transcripts and proteins, while further experiments could more disambiguate the source of post-transcriptional regulation.

\section{Materials and Methods}

\subsection{Processing and alignment of input data}

The processed microarray relative abundance data that was used to generate Figure 2 of Brauer et al. 2008 was downloaded from http://growthrate.princeton.edu \cite{Brauer:2008jn}.  This dataset contains the expression of 5537 transcripts across 36 growth conditions (6 limiting nutrients x 6 growth rates) relative to a standard fast growing carbon-limited reference condition. Proteomics data was drawn from the Hackett proteomics dataset discussed in Chapter \ref{ch:simmer}. This dataset contains the abundance of 1187 proteins across 25 growth conditions (5 limiting nutrients x 5 growth rates) relative to a common slow-growing phosphate-limited chemostat. Among the measured proteins, 1161 proteins matched to a unique gene and were chosen for subsequent analysis and missing values were imputed using knn-imputation \cite{Troyanskaya:2001uh}. The five limiting nutrients shared by these datasets are (glucose (C), ammonium (N), phosphate (P), leucine (L) and uracil (U) limitation, with isogenic ura3$^{-}$ or leu2$^{-}$ auxotrophic strains being used during uracil and leucine limitation respectively. Media formulation was identical except only 5 g/l glucose was for non-carbon limitation in Brauer et al. 2008, while 21 or 22 g/l glucose was used in the Hackett study. The strains used in Brauer et al. 2008 were derived from CEN.PK parental strain \cite{vanDijkenJP:2000er}, while those in the Hackett experiment were derived from a S288C parental strain \cite{Winston:1995io}.

Because transcript abundance was assessed at 6 growth rates per limiting nutrient, while proteins were only measured at 5 growth rates, the growth rates in these two studies are not equivalent. To adjust for differences in growth rate, on a gene-wise basis, the expected abundance of transcripts at each proteomic's growth rate was estimated through linear interpolation (Equation \ref{eq:transc_interp}). Here, DR$_{P}$ is a proteomics dilution rate, and DR$_{T}^{+}$ and DR$_{T}^{-}$ are the dilution rates closest to DR$_{P}$ (where DR$_{T}^{-}$ $<$ DR$_{T}^{+}$). Based on these dilution rates, T$^{*}$ is the expected transcript abundance at DR$_{P}$, weighting transcript abundances, T$^{-}$ and T$^{+}$. As most transcriptional variability is linear within a given limiting nutrient \cite{Brauer:2008jn}, this procedure should accurately estimate transcript abundance for the vast majority of observations. 

\begin{align}
\Delta &= \left(DR_{P} - DR_{T}^{-}\right) / \left(DR_{T}^{+} - DR_{T}^{-}\right)\notag\\
T^{*} &= (1-\Delta)T^{-} + \Delta T^{+}\label{eq:transc_interp}
\end{align}

\subsection{Testing the correlation between protein and transcript relative abundances}

To make protein and transcript abundances generally comparable, features in each dataset were mean centered. This allowed for direct testing of whether changes in transcript abundance were similarly reflected in changes in protein abundance.  To determine whether departures between proteins and transcripts could be explained by simple variation in growth rate between the conditions, for each gene, a per-gene value of $\alpha_{i}$ and $\lambda_{i}$ was found that maximizes the correlation between P$_{iJ}$ and T$_{iJ}$ + $\sfrac{\alpha_{i}}{\left(\lambda_{i} + GR_{J}\right)}$.  These adjusted transcript abundances were again mean centered before assessing overall correlations between changes in transcript and protein abundance. To test whether the transcript and protein abundances of a single gene are significantly correlated, 1000 null spearman correlations were generated by permuting protein abundances with respect to transcript abundances and calculating two-tailed significance (Equation \ref{ch-pta-permpval}). To allow for differences in the fraction of genes where transcripts and proteins are positively versus negatively correlated, significance at a false discovery rate of 0.05 was calculated separately for genes where transcripts and proteins are positively and negatively correlated \cite{Storey:2003cj}.

\begin{align}
p_{i} = 1 - 2\left|0.5 - \frac{\sum_{p = 1}^{1000}r^{spear}_{i} > r_{ip}^{spear-null}}{1000}\label{ch-pta-permpval}\right|
\end{align}

To determine the major sources of variation in proteins-per-transcript and the original proteomics and transcriptomics data, each dataset was analyzed using Jackstraw to estimate, the number of significant principle components, to estimate the principle components, and to estimate sparse loadings of features \cite{Chung:2015bq}. As these loadings are only non-zero when a feature's variation is impacted by a principle component, if we assume that the principle components are a surrogate for post-transcriptional regulation, then the jackstraw projection of the principle components is an appropriate summary of the impact of post-transcriptional regulation.

\subsection{Identifying gene sets associated with variation in proteins-per-transcript}

GSEA

voronoi tesselation